%%%%%%%%%%%%%%%%%%%% author.tex %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%
% sample root file for your "contribution" to a contributed volume
%
% Use this file as a template for your own input.
%
%%%%%%%%%%%%%%%% Springer %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


% RECOMMENDED %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\documentclass[graybox]{svmult}

% choose options for [] as required from the list
% in the Reference Guide

\usepackage{mathptmx}       % selects Times Roman as basic font
\usepackage{helvet}         % selects Helvetica as sans-serif font
\usepackage{courier}        % selects Courier as typewriter font
\usepackage{type1cm}        % activate if the above 3 fonts are
                            % not available on your system
%
\usepackage{makeidx}         % allows index generation
\usepackage{graphicx}        % standard LaTeX graphics tool
                             % when including figure files
\usepackage{multicol}        % used for the two-column index
\usepackage[bottom]{footmisc}% places footnotes at page bottom

%%% Personal packages
%\usepackage[utf8]{inputenc}

% see the list of further useful packages
% in the Reference Guide

\makeindex             % used for the subject index
                       % please use the style svind.ist with
                       % your makeindex program

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{document}
\tableofcontents

\title*{non-protein-coding RNAs as regulators of development in tunicates}
% Use \titlerunning{Short Title} for an abbreviated version of
% your contribution title if the original one is too long
\author{Author A and Author B}
% Use \authorrunning{Short Title} for an abbreviated version of
% your contribution title if the original one is too long
\institute{Author Name \at Name, Address of Institute, \email{writeemail@com}
\and   Author Name \at Name, Address of Institute \email{writeemail@com}}
%
% Use the package "url.sty" to avoid
% problems with special characters
% used in your e-mail or web address
%
\maketitle

\abstract*{Each chapter should be preceded by an abstract (10--15 lines long).}

\abstract{Each chapter should be preceded by an abstract (10--15 lines long)}



\section{Introduction}
\label{sec:1}

The function of noncoding RNAs in tunicates development dated earliest in the 90«s from the works of Swalla \& Jeffry in which RNAs localized in the yellow crescent or myoplasm, a cytoskeletal domain in oocytes of the ascidian \textit{Styela clava} were discovered \cite{Swalla1995}. This  yellow crescent or YC RNA identified to be present throughout embryonic development was the first example involved in envisioning the future of a growing family of ncRNAs that would play important roles in growth and development in tunicates\cite{Swalla1995}. This asymmetrically distributed ascidian RNAs were part of the set of many other RNAs known as maternally synthesized cytoplasmically localized RNAs, which were discovered first in oocytes of Xenopus \cite{Bashirullah1998}

\section{miRNA families origin and evolutionary perspective}
\label{sec:2}

\subsection{miRNA identification and validation}

The first microRNA (miRNA) in tunicates was discovered in the year 2000 from the work of \textit{Pasquinelli et al., 2000} when they were studying the temporal regulation of let-7 during development by using samples of small RNAs of a wide range of animal species, in which the ascidian \textit{Ciona intestinalis} was included as well as other vertebrates, hemichordates, mollusc, annelids, arthropod and other bilateral and nonbilateria animals \cite{Pasquinelli2000}. Later on the year 2003  the same team suggested that let-7 RNA may control the late temporal transitions during development across animal phylogeny \cite{Pasquinelli2003} albeit it was not identified on basal metazoans such as cnidarians and poriferans. 

Then after the era of genome sequencing became available, it was launched in 2005 the computational screening of whole-genomes of non-model organism as tunicates. Beginning with the Cionas \textit{C. intestinalis} and \textit{C. savigni} a profile-based strategy was implemented in the ERPIN program \cite{Legendre2005}. On that work were detected a set of new miRNAs candidates considered as \textit{C. intestinalis} specific such as the members of the family miR-9 and miR-79 and as it was expected, other miRNA families were found homologous between both Cionas like the families miR-124;92;98;325;310-313 and let-7. Coincidentally, by the same year a whole-genomic comparative approach in the urochordate lineage was performed on the species \textit{C. intestinalis, C. savignyi} and \textit{O. dioica}. Using a computational screening of structured ncRNAs based upon homology between predicted precursor hairpin structures  41 miRNA candidates were detected including let-7 and other six known candidates in \textit{C. intestinalis} \cite{Missal2005}. After all, the same group in 2007 implemented a structure-based clustering approach in \textit{C. intestinalis} predicted 58 miRNAs, of which only let-7, miR-7, miR-124, and miR-126 coincided with the previously annotated miRNAs \cite{Will2007}. 

Thus far, the primary focus to identify miRNAs into urochordate linaje has been mainly toward the use of computational approaches but soon came up the use of new hybrid strategies combining computational and experimental studies to validate candidate families previously detected. For instance the first bona fide record for \textit{C. intestinalis} was registered in mirBase only in its Release 11. Those first miRNAs records were derived from the work published in 2007 by Norden-Krichmar et al., \cite{Norden-Krichmar2007}. The authors searched for conservation with the seed region of the known mature miRNA sequences from miRBase release 2006 on the whole-genomic sequences of \textit{C. intestinalis} and \textit{C. savignyi}. Those miRNAs were aligned locally using the FASTA/ssearch34 program. Only matches of 90\% identity or better were retained. In further steps these authors studied RNA sequences that folds like hairpin structures with the mature miRNA sequence in the stem region including other typical features exhibit in miRNA hairpins. By manual curation of the genomic sequences predicted by the software mfold which folded like hairpin structures, a set of 18 miRNA molecules were detected which appeared conserved in both Cionas. After all, using  Northern blot analyses in the adult tissue of \textit{C. intestinalis} the authors confirmed expression of  let-7, miR-7, and miR-126, as well as 11 other conserved miRNA families.


Until 2008, most of the miRNAs annotations were concentrated in Cionas, but new annotation approaches for other species in tunicates were appearing slowly to increase then the repertory of new miRNAs families in urochordates. In this order of ideas, the first repertory of miRNAs based on non-Cionas species was published by Fu et al., in  2008 for the larvacean \textit{O. dioica} \cite{Fu2008}. At that time the authors were studying the temporal-spatial expression patterns of conserved miRNAs in different developmental stages of oocytes, 1-cell zygote, 2-8 cell embryons, blastulas, gastrulas, tadpoles (in different stages) and animals from 1 to 6 days from \textit{O. dioica}. In this research, small RNAs were isolated, amplified by RT-PCR and rapid amplification of cDNA ends (RACE) of the developmental stages, cloned and sequenced. Blast searches using the sequences of cloned small RNA libraries were used to annotate small RNAs as miRNA candidates. In further steps the recovered  genomic flanking sequences each side of those mapped candidates were used as input to predicted secondary structures by mfold v3.1. This step was used to detect candidates that folds like miRNA hairpins  and aimed to decrease the set of false positive potential miRNAs in \textit{O. dioica}. Finally, for this set of potential candidates a developmental miRNA array dot blot analyses were performed to detect miRNA expression. With this approach from 3.066 sequenced small RNA clones only for 55 miRNAs was detected expression. As a conclusion the authors suggested that those candidates were expressed throughout the short life cycle of \textit{O. dioica} showing that some of them were stocked as maternal determinants prior to rapid embryonic development.  Besides the authors identified a set of sex-specific miRNAs that appeared as male/female gonad differentiation which became apparent and was maintained throughout spermatogenesis \cite{Fu2008}. Unexpectedly, the majority of the miRNAs loci in \textit{O. dioica} were located in antisense orientations into the hosted genes in oposite fashion observed in the majority of the known mammalian miRNAs at that time. 

Between the years 2009 and 2015 the majority of the studies of miRNAs in tunicates were focused into the validation of expression of computational predicted miRNAs in Cionas specially focused in \textit{C. intestinalis} as model organism of tunicates or into the test of new computational approaches as miRTRAP, miRDeep2 and miRRim2 which used next- generation sequencing libraries of small RNAs derived from \textit{C. intestinalis} to validate their algorithms. Then by the year 2016 the first comparative homology based search strategy let us to identify the repertory on miRNAs and other ncRNAs in the carpet sea squirt \textit{Didemnum vexillum} with a preliminary comparative analysis of gain and losses of miRNA families on chordates which included the \textit{Cionas, O. dioca} and the colonial tunicate \textit{Botryllus schlosseri} \cite{velandia:2016}. By the same year, from the preliminary genome sequence assembled for the Southern Ocean salp, \textit{Salpa thompsoni} (Urochordata, Thaliacea) a set of miRNAs families were detected \cite{Jue2016} and in 2017 the prediction of miRNAs families were reported to the species \textit{Halocynthia roretzi}. On the following two sections we will focus on those stages of the fascinated increased screening of the miRNA repertory in tunicates.

\subsubsection{Validation and detection of miRNA families in Cionas in this decade}

At the end of the last decade the application of next generation sequencing technologies to sequence small RNA libraries changed the common way used to detect expression of miRNAs in many organisms including the tunicates. This technology became in one of the most common approaches that supported methods like RT-PCR, microarrays or dot blotting which were previously used to validate miRNA expression in tunicates. In 2009 after preparing small RNA libraries from various developmental stages including unfertilized eggs, early
embryos, late embryos and adults from  \textit{C. intestinalis} was performed high-throughput sequencing of cDNA with an Illumina 1G Genome Analyzer experiments. These sequencing led to document 80 miRNAs families for  \textit{C. intestinalis}. Unexpectedly, were detected a distinct species of small RNAs processed outside of the miRNA precursors which were termed as moRs or miRNA-offset RNAs \cite{Shi2009}. Later on, after extracting non-coding conserved regions of whole genome alignments between  \textit{C. intestinalis} and \textit{C. savigny} a set of 12 million sequences were computationally folded using RNAfold and mfold. Then after combining the following criteria: structure/sequence conservation, homology to known miRNAs, and phylogenetic footprinting the authors detected 
a set of 458 candidate sequences \cite{Keshavan2010}. Then in order to validate those candidate, RT-PCR and PAGE were conducted to design a custom microarray. After screened them for miRNA expression were identifying that 244 of the 458 miRNA predictions were represented either in their microarray data or in the Illumina  database constructed previously for small RNA derived from \textit{C. intestinalis} by \cite{Shi2009}. Although they failed to predict 39 previously characterized miRNAs, it was suggested in this work that \textit{C. intestinalis} genome may encode about 300 miRNA genes. Then to increase the miRNAs collection in \textit{C. intestinalis} a novel computational strategy for the systematic, whole-genome identification of microRNA from high throughput sequencing information was developed in 2010 by \cite{Hendrix2010}. That method, known as miRTRAP, incorporated not only the mechanisms of microRNA biogenesis but also includes additional criteria regarding the prevalence and quality of small RNAs arising from the antisense strand and the neighboring loci. With that approach, nearly 400 putative microRNAs loci were detected. In short words these strategy relies on the way how the the biochemical machinery processes pre-miRNA hairpins produces short RNA products. This approach is highly depended on the deep of the small RNAs mapped to a given locus and is highlighted by the authors that the approach requires an accurate assignment of small RNA sequences on their relative positions along the hairpin, that is, miR/miR*, moR/moR* and loop \cite{Hendrix2010}. Again a new approach took advantage of importance to detect miRNAs from the high-throughput sequencing of small RNAs available from \cite{Shi2009}. These approach known as  miRDeep2 improved the algorithm of its first version  miRDeep \cite{Friedlander2012} and let to identify with an accuracy of 98.6â€“99.9\% canonical and non-canonical miRNAs in different species. These approach reported 313 known and 127 novel ones miRNAs in \textit{C. intestinalis}. 
In the same year the program miRRim2 \cite{Terai2012} was applied to the \textit{C. intestinalis} genome, in which some candidates identified from the work of \cite{Hendrix2010} and the several promising candidates were detected. In 2013, \cite{Kusakabe2013} was investigated the expression patterns of the cluster miR-1 and miR-133 in \textit{C. intestinalis} and in \textit{C. savignyi}. RT-PCR amplification of miR-1/133 precursors were performed and PCR products were subcloned and sequenced. Whole-mount in situ hybridization to detect cin-miR-1/miR-133 primary transcript was performed and LNA Northern blotting was conducted on different developmental stages. 


\subsubsection{The new era to get deep insights into the repertory of miRNA in other urochordates}

Since 2016 new approximations has increased our knowledge about new families in other tunicates thanks to the sequence of new urochordate genomes of the species \textit{D. vexillum}, \textit{S. thompsoni} and \textit{H. roretzi}. Please summary of our Dvexillum paper \cite{velandia:2016} including the first reported preliminary annotation for colonial tunicate \textit{B. schlosseri} beside the one for Dvexillum. For the preliminary assembled of the genome sequence for the Southern Ocean salp \textit{S. thompsoni} \cite{Jue2016} were  small RNA libraries constructed to be sequenced on an Illumina Hiseq 2000. After filtering data sets to 18--24 nt for miRNA and 28--32 nt for piRNA, the reads were aligned to \textit{S. thompsoni} genome and miRNA gene folding predictions were performed using RNAfold. In this initial survey of small RNAs, were revealed the presence of known, conserved miRNAs, as well as novel miRNA genes and mature miRNA signatures for varying developmental stages. Then in 2017, the prediction of 319 miRNAs candidates in  \textit{H. roretzi} were obtained through three complementary methods. The experimental validation suggested that more than half of these candidate miRNAs are expressed during embryogenesis. The expression of some of the predicted miRNAs were validated by RT-PCR using embryonic RNA. In this approach \textit{C. robusta}  small RNA-Seq reads derived from \textit{C. robusta} \cite{Shi2009} (previously known as  \textit{C. intestinalis} today reclassified) was used to identify conserved miRNAs in \textit{H. roretzi} \cite{Wang2017} .  

% BibTeX users please use
\bibliographystyle{plain}
\bibliography{tuni.bib}
%%%%%%

\end{document}
