<?xml version="1.0" encoding="iso-8859-1" ?> 
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" 
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">  
<!--http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd-->  
<html xmlns="http://www.w3.org/1999/xhtml"  
> 
<head><title>non-protein-coding RNAs as regulators of development in tunicates</title> 
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" /> 
<meta name="generator" content="TeX4ht (http://www.tug.org/tex4ht/)" /> 
<meta name="originator" content="TeX4ht (http://www.tug.org/tex4ht/)" /> 
<!-- xhtml,html --> 
<meta name="src" content="tunicates_ncrna.tex" /> 
<meta name="date" content="2018-03-04 22:45:00" /> 
<link rel="stylesheet" type="text/css" href="tunicates_ncrna.css" /> 
</head><body 
>
  <h3 class="likesectionHead"><a 
 id="x1-1000"></a>Contents</h3>
  <div class="tableofcontents">
  &#x00A0;<span class="sectionToc" >1 <a 
href="#x1-20001" id="QQ2-1-4">Introduction</a></span>
<br />  &#x00A0;<span class="sectionToc" >2 <a 
href="#x1-30002" id="QQ2-1-5">miRNA families origin and evolutionary perspective</a></span>
<br />  &#x00A0;&#x00A0;<span class="subsectionToc" >2.1 <a 
href="#x1-40002.1" id="QQ2-1-6">Origins and Evolution of MicroRNAs</a></span>
<br />  &#x00A0;&#x00A0;<span class="subsectionToc" >2.2 <a 
href="#x1-50002.2" id="QQ2-1-7">miRNA identification and validation</a></span>
<br />  &#x00A0;&#x00A0;<span class="subsectionToc" >2.3 <a 
href="#x1-80002.3" id="QQ2-1-10">miRNA in clusters</a></span>
<br />  &#x00A0;<span class="sectionToc" >3 <a 
href="#x1-90003" id="QQ2-1-14">miRNAs and its role in development</a></span>
<br />  &#x00A0;&#x00A0;<span class="subsectionToc" >3.1 <a 
href="#x1-100003.1" id="QQ2-1-15">miRNAs discovery and its role in development</a></span>
<br />  &#x00A0;&#x00A0;<span class="subsectionToc" >3.2 <a 
href="#x1-110003.2" id="QQ2-1-16">Neuronal fate determination and regulation by miR-124</a></span>
<br />  &#x00A0;<span class="sectionToc" >4 <a 
href="#x1-150004" id="QQ2-1-20">Other ncRNAs associated with development</a></span>
<br />  &#x00A0;&#x00A0;<span class="subsectionToc" >4.1 <a 
href="#x1-160004.1" id="QQ2-1-21">Yellow Crescent RNA</a></span>
<br />  &#x00A0;&#x00A0;<span class="subsectionToc" >4.2 <a 
href="#x1-170004.2" id="QQ2-1-22">MicroRNA-offset RNAs</a></span>
<br />  &#x00A0;&#x00A0;<span class="subsectionToc" >4.3 <a 
href="#x1-180004.3" id="QQ2-1-23">Long Noncoding RNA RMST</a></span>
<br />  &#x00A0;&#x00A0;<span class="subsectionToc" >4.4 <a 
href="#x1-190004.4" id="QQ2-1-24">Splices-leader RNA</a></span>
<br />  &#x00A0;<span class="sectionToc" >5 <a 
href="#x1-200005" id="QQ2-1-25">Acknowledgments</a></span>
<br />  &#x00A0;<span class="sectionToc" ><a 
href="#Q1-1-27">References</a></span>
  </div>
  <div class="maketitle">
                                                               

                                                               
                                                               

                                                               
                                                               

                                                               
                                                               

                                                               

<h2 class="titleHead">non-protein-coding
RNAs
as
regulators
of
development
in
tunicates</h2>
<div class="author" >Cristian
A.
Velandia-Huerto*,
Federico
D.
Brown,
Adriaan
A.
Gittenberger,
Peter
F.
Stadler
and
Clara
I.
Bermúdez-Santana</div>
                                                               

                                                               
<a 
 id="x1-1001f0"></a><span class="footnote-mark"><a 
 id="fn0x1">  </a></span><span 
class="ptmr7t-x-x-85">C.A.</span>
<span 
class="ptmr7t-x-x-85">Velandia-Huerto,</span><br />
<span 
class="ptmr7t-x-x-85">Bioinformatics</span>
<span 
class="ptmr7t-x-x-85">Group,</span>
<span 
class="ptmr7t-x-x-85">Department</span>
<span 
class="ptmr7t-x-x-85">of</span>
<span 
class="ptmr7t-x-x-85">Computer</span>
<span 
class="ptmr7t-x-x-85">Science,</span>
<span 
class="ptmr7t-x-x-85">and</span>
<span 
class="ptmr7t-x-x-85">Interdisciplinary</span>
<span 
class="ptmr7t-x-x-85">Center</span>
<span 
class="ptmr7t-x-x-85">for</span>
<span 
class="ptmr7t-x-x-85">Bioinformatics,</span>
<span 
class="ptmr7t-x-x-85">Universit</span><span 
class="ptmr7t-x-x-85">ät</span>
<span 
class="ptmr7t-x-x-85">Leipzig,</span>
<span 
class="ptmr7t-x-x-85">H</span><span 
class="ptmr7t-x-x-85">ärtelstraße</span>
<span 
class="ptmr7t-x-x-85">16&#8211;18,</span>
<span 
class="ptmr7t-x-x-85">D-04107,</span>
<span 
class="ptmr7t-x-x-85">Leipzig,</span>
<span 
class="ptmr7t-x-x-85">Germany.</span><br />
<span 
class="ptmr7t-x-x-85">Biology</span>
<span 
class="ptmr7t-x-x-85">Department,</span>
<span 
class="ptmr7t-x-x-85">Universidad</span>
<span 
class="ptmr7t-x-x-85">Nacional</span>
<span 
class="ptmr7t-x-x-85">de</span>
<span 
class="ptmr7t-x-x-85">Colombia,</span>
<span 
class="ptmr7t-x-x-85">Carrera</span>
<span 
class="ptmr7t-x-x-85">45</span>
<span 
class="ptmr7t-x-x-85">#</span>
<span 
class="ptmr7t-x-x-85">26-85,</span>
<span 
class="ptmr7t-x-x-85">Edif.</span>
<span 
class="ptmr7t-x-x-85">Uriel</span>
<span 
class="ptmr7t-x-x-85">Guti</span><span 
class="ptmr7t-x-x-85">érrez,</span>
<span 
class="ptmr7t-x-x-85">Bogot</span><span 
class="ptmr7t-x-x-85">á,</span>
<span 
class="ptmr7t-x-x-85">Colombia,</span>
<span 
class="ptmr7t-x-x-85">*Correspondence</span>
<span 
class="ptmr7t-x-x-85">e-mail:</span>
<span 
class="ptmr7t-x-x-85">cristian@bioinf.uni-leipzig.de.</span><br />
<span 
class="ptmr7t-x-x-85">F.D.</span>
<span 
class="ptmr7t-x-x-85">Brown,</span><br />
<span 
class="ptmr7t-x-x-85">Departamento</span>
<span 
class="ptmr7t-x-x-85">de</span>
<span 
class="ptmr7t-x-x-85">Zoologia,</span>
<span 
class="ptmr7t-x-x-85">Instituto</span>
<span 
class="ptmr7t-x-x-85">Bioci</span><span 
class="ptmr7t-x-x-85">ências,</span>
<span 
class="ptmr7t-x-x-85">Universidade</span>
<span 
class="ptmr7t-x-x-85">de</span>
<span 
class="ptmr7t-x-x-85">S</span><span 
class="ptmr7t-x-x-85">ão</span>
<span 
class="ptmr7t-x-x-85">Paulo,</span>
<span 
class="ptmr7t-x-x-85">Rua</span>
<span 
class="ptmr7t-x-x-85">do</span>
<span 
class="ptmr7t-x-x-85">Mat</span><span 
class="ptmr7t-x-x-85">ão,</span>
<span 
class="ptmr7t-x-x-85">Tr.</span>
<span 
class="ptmr7t-x-x-85">14</span>
<span 
class="ptmr7t-x-x-85">no.</span>
<span 
class="ptmr7t-x-x-85">101,</span>
<span 
class="ptmr7t-x-x-85">S</span><span 
class="ptmr7t-x-x-85">ão</span>
<span 
class="ptmr7t-x-x-85">Paulo,</span>
<span 
class="ptmr7t-x-x-85">SP,</span>
<span 
class="ptmr7t-x-x-85">Brazil</span><br />
<span 
class="ptmr7t-x-x-85">Laboratorio</span>
<span 
class="ptmr7t-x-x-85">de</span>
<span 
class="ptmr7t-x-x-85">Biolog</span><span 
class="ptmr7t-x-x-85">ía</span>
<span 
class="ptmr7t-x-x-85">del</span>
<span 
class="ptmr7t-x-x-85">Desarrollo</span>
<span 
class="ptmr7t-x-x-85">Evolutiva,</span>
<span 
class="ptmr7t-x-x-85">Departamento</span>
<span 
class="ptmr7t-x-x-85">de</span>
<span 
class="ptmr7t-x-x-85">Ciencias</span>
<span 
class="ptmr7t-x-x-85">Biol</span><span 
class="ptmr7t-x-x-85">ógicas,</span>
<span 
class="ptmr7t-x-x-85">Universidad</span>
<span 
class="ptmr7t-x-x-85">de</span>
<span 
class="ptmr7t-x-x-85">los</span>
<span 
class="ptmr7t-x-x-85">Andes,</span>
<span 
class="ptmr7t-x-x-85">Cra</span>
<span 
class="ptmr7t-x-x-85">1</span>
<span 
class="ptmr7t-x-x-85">No.</span>
<span 
class="ptmr7t-x-x-85">18A-12,</span>
<span 
class="ptmr7t-x-x-85">Bogot</span><span 
class="ptmr7t-x-x-85">á,</span>
<span 
class="ptmr7t-x-x-85">Colombia,</span>
<span 
class="ptmr7t-x-x-85">e-mail:</span>
<span 
class="ptmr7t-x-x-85">fdbrown@usp.br.</span>
<span 
class="ptmr7t-x-x-85">A.A.</span>
<span 
class="ptmr7t-x-x-85">Gittenberger,</span><br />
<span 
class="ptmr7t-x-x-85">Institute</span>
<span 
class="ptmr7t-x-x-85">of</span>
<span 
class="ptmr7t-x-x-85">Biology,</span>
<span 
class="ptmr7t-x-x-85">Leiden</span>
<span 
class="ptmr7t-x-x-85">University,</span>
<span 
class="ptmr7t-x-x-85">P.O.</span>
<span 
class="ptmr7t-x-x-85">Box</span>
<span 
class="ptmr7t-x-x-85">9505,</span>
<span 
class="ptmr7t-x-x-85">2300</span>
<span 
class="ptmr7t-x-x-85">RA,</span>
<span 
class="ptmr7t-x-x-85">Leiden,</span>
<span 
class="ptmr7t-x-x-85">Netherlands</span><br />
<span 
class="ptmr7t-x-x-85">GiMaRIS,</span>
<span 
class="ptmr7t-x-x-85">BioScience</span>
<span 
class="ptmr7t-x-x-85">Park</span>
<span 
class="ptmr7t-x-x-85">Leiden,</span>
<span 
class="ptmr7t-x-x-85">J.H.</span>
<span 
class="ptmr7t-x-x-85">Oortweg</span>
<span 
class="ptmr7t-x-x-85">21,</span>
<span 
class="ptmr7t-x-x-85">2333</span>
<span 
class="ptmr7t-x-x-85">CH,</span>
<span 
class="ptmr7t-x-x-85">Leiden,</span>
<span 
class="ptmr7t-x-x-85">Netherlands</span><br />
<span 
class="ptmr7t-x-x-85">Naturalis</span>
<span 
class="ptmr7t-x-x-85">Biodiversity</span>
<span 
class="ptmr7t-x-x-85">Center,</span>
<span 
class="ptmr7t-x-x-85">Darwinweg</span>
<span 
class="ptmr7t-x-x-85">2,</span>
<span 
class="ptmr7t-x-x-85">2333</span>
<span 
class="ptmr7t-x-x-85">CR,Leiden,</span>
<span 
class="ptmr7t-x-x-85">Netherlands,</span>
<span 
class="ptmr7t-x-x-85">e-mail:</span>
<span 
class="ptmr7t-x-x-85">gittenberger@gimaris.com.</span><br />
<span 
class="ptmr7t-x-x-85">P.F.</span>
<span 
class="ptmr7t-x-x-85">Stadler,</span><br />
<span 
class="ptmr7t-x-x-85">Bioinformatics</span>
<span 
class="ptmr7t-x-x-85">Group,</span>
<span 
class="ptmr7t-x-x-85">Department</span>
<span 
class="ptmr7t-x-x-85">of</span>
<span 
class="ptmr7t-x-x-85">Computer</span>
<span 
class="ptmr7t-x-x-85">Science,</span>
<span 
class="ptmr7t-x-x-85">and</span>
<span 
class="ptmr7t-x-x-85">Interdisciplinary</span>
<span 
class="ptmr7t-x-x-85">Center</span>
<span 
class="ptmr7t-x-x-85">for</span>
<span 
class="ptmr7t-x-x-85">Bioinformatics,</span>
<span 
class="ptmr7t-x-x-85">Universit</span><span 
class="ptmr7t-x-x-85">ät</span>
<span 
class="ptmr7t-x-x-85">Leipzig,</span>
<span 
class="ptmr7t-x-x-85">H</span><span 
class="ptmr7t-x-x-85">ärtelstraße</span>
<span 
class="ptmr7t-x-x-85">16&#8211;18,</span>
<span 
class="ptmr7t-x-x-85">D-04107,</span>
<span 
class="ptmr7t-x-x-85">Leipzig,</span>
<span 
class="ptmr7t-x-x-85">Germany,</span>
<span 
class="ptmr7t-x-x-85">e-mail:</span>
<span 
class="ptmr7t-x-x-85">studla@bioinf.uni-leipzig.de.</span><br />
<span 
class="ptmr7t-x-x-85">C.I.</span>
<span 
class="ptmr7t-x-x-85">Berm</span><span 
class="ptmr7t-x-x-85">údez-Santana,</span><br />
<span 
class="ptmr7t-x-x-85">Biology</span>
<span 
class="ptmr7t-x-x-85">Department,</span>
<span 
class="ptmr7t-x-x-85">Universidad</span>
<span 
class="ptmr7t-x-x-85">Nacional</span>
<span 
class="ptmr7t-x-x-85">de</span>
<span 
class="ptmr7t-x-x-85">Colombia,</span>
<span 
class="ptmr7t-x-x-85">Carrera</span>
<span 
class="ptmr7t-x-x-85">45</span>
<span 
class="ptmr7t-x-x-85">#</span>
<span 
class="ptmr7t-x-x-85">26-85,</span>
<span 
class="ptmr7t-x-x-85">Edif.</span>
<span 
class="ptmr7t-x-x-85">Uriel</span>
<span 
class="ptmr7t-x-x-85">Guti</span><span 
class="ptmr7t-x-x-85">érrez,</span>
<span 
class="ptmr7t-x-x-85">Bogot</span><span 
class="ptmr7t-x-x-85">á,</span>
<span 
class="ptmr7t-x-x-85">Colombia,</span>
<span 
class="ptmr7t-x-x-85">e-mail:</span>
<span 
class="ptmr7t-x-x-85">cibermudezs@unal.edu.co.</span>


                                                               
<a 
 id="Q1-1-2"></a>
<a 
 id="Q1-1-3"></a>
  </div>
<!--l. 128--><p class="noindent" ><span 
class="ptmb7t-">Abstract </span>Tunicates, or urochordates, are a group of small marine organisms that are found
widely throughout the seas of the world. As most plausible sister group of the vertebrates
they are of utmost importance for a comprehensive understanding of chordate evolution,
hence they have served as model organisms for many aspects of the developmental
biology. Current genomic analysis of tunicates indicates that their genomes evolved with a
fast rate not only at the level of nucleotide substitutions but also in terms of genomic
organization. The latter involves genome reduction, rearrangements, as well as the loss
of some important coding and non-coding RNA (ncRNAs) elements and even
entire genomic regions that are otherwise well conserved. These observations
are largely based on evidence from comparative genomics resulting from the
analysis of well-studied gene families such as the Hox genes and their non-coding
elements. In this chapter the focus lies on the ncRNA complement of tunicates,
with a particular emphasis on microRNAs, which have already been studied
extensively for other animal clades. MicroRNAs are known as important regulators of
key genes in animal development and they are intimately related to the increase
morphological complexity in higher metazoans. Here we review the discovery, evolution,
and genome organization of the miRNA repertoire, which has been drastically
reduced and restructured in tunicates compared to the chordate ancestor. Known
functions of microRNAs as regulators of development in tunicates are a central
topic. For instance we consider the role of miRNAs as regulators of the muscle
development and their importance in the regulation of the differential expression
during the oral siphon regeneration. Beyond microRNAs, we touch upon the
functions of some other ncRNAs such as Yellow Crescent RNA, moRNAs, RMST
lncRNAs, or spliced-leader (SL) RNAs, which have diverse functions associated with
the embryonic development, neurogenesis and mediation of mRNA stability in
general.
</p>
  <h3 class="sectionHead"><span class="titlemark">1  </span> <a 
 id="x1-20001"></a>Introduction</h3>
<!--l. 133--><p class="noindent" >Tunicates are organisms characterized by a fast rate of genomic and developmental
evolution. Some fast evolving evolutionary changes include loss of synteny, fast changes
in cis-regulatory sequences, and loss of several key regulatory developmental genes (<a 
href="#XSatou2008">Satou
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSatou2008">2008</a>;&#x00A0;<a 
href="#XDenoeud1381">Denoeud et&#x00A0;al.</a>,&#x00A0;<a 
href="#XDenoeud1381">2010</a>), such as several central or posterior Hox genes
involved in AP patterning of metazoans (<a 
href="#XIkuta:2005">Ikuta and Saiga</a>,&#x00A0;<a 
href="#XIkuta:2005">2005</a>) and Gbx involved
in the establishment of the midbrain-hindbrain boundary in vertebrates (<a 
href="#XYagi2003">Yagi
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XYagi2003">2003</a>).
</p><!--l. 141--><p class="indent" >  The role of noncoding RNAs in tunicate development has been studied since the
1990&#8217;s, dating back to the seminal work by Swalla &amp; Jeffry on the RNAs localized in the
yellow crescent or myoplasm, a cytoskeletal domain in oocytes of the ascidian <span 
class="ptmri7t-">Styela</span>
<span 
class="ptmri7t-">clava </span>(<a 
href="#XSwalla1995">Swalla and Jeffery</a>,&#x00A0;<a 
href="#XSwalla1995">1995</a>). The yellow crescent (YC) RNA identified to
be present throughout embryonic development was the first example of what
is now a large and rapidly growing class of ncRNAs with important roles in
                                                               

                                                               
growth and development in tunicates. This asymmetrically distributed ascidian
RNAs were part of the set of many other RNAs known as maternally synthesized
cytoplasmically localized RNAs, discovered first in oocytes of Xenopus (<a 
href="#XBashirullah1998">Bashirullah
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBashirullah1998">1998</a>).
</p><!--l. 152--><p class="indent" >  The current state of knowledge on ncRNAs in tunicates is far from comprehensive and
complete. Nevertheless, in particular microRNAs have been studied already in
some detail in several tunicates species, such as, <span 
class="ptmri7t-">Oikopleura dioica</span>, both Ciona
species<span class="footnote-mark"><a 
href="tunicates_ncrna2.html#fn1x0"><sup class="textsuperscript">1</sup></a></span><a 
 id="x1-2001f1"></a> ,
<span 
class="ptmri7t-">Didemnum vexillum</span>, and <span 
class="ptmri7t-">Salpa thompsoni</span>. These studies have revealed many losses of
miRNA families that are very well conserved outside the tunicates. At the same time,
many gains of unique miRNAs among recently divergent lineages in the tunicates when
compared to other groups of chordates (<a 
href="#XFu:08">Fu et&#x00A0;al.</a>,&#x00A0;<a 
href="#XFu:08">2008</a>;&#x00A0;<a 
href="#XVelandia-Huerto2016">Velandia-Huerto et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVelandia-Huerto2016">2016</a>).
Relaxed constraints in the evolution of genomes and developmental trajectories in the
tunicates may have been responsible for the plethora of reproductive strategies,
morphologies, and life histories observed in the group (<a 
href="#XHolland2015">Holland</a>,&#x00A0;<a 
href="#XHolland2015">2015</a>).
</p>
  <h3 class="sectionHead"><span class="titlemark">2  </span> <a 
 id="x1-30002"></a>miRNA families origin and evolutionary perspective</h3>
<!--l. 173--><p class="noindent" >
</p>
  <h4 class="subsectionHead"><span class="titlemark">2.1  </span> <a 
 id="x1-40002.1"></a>Origins and Evolution of MicroRNAs</h4>
<!--l. 175--><p class="noindent" >MicroRNAs (miRNAs) have been described in almost all animals and plants as well as
unicellular eukaryotes. They are important post-transcriptional regulators of gene
expression affecting a sizable fraction of all mRNAs (<a 
href="#XAmeres:13">Ameres and Zamore</a>,&#x00A0;<a 
href="#XAmeres:13">2013</a>).
Mechanistically, miRNAs depends on the presence of the evolutionarily even older
RNA interference pathways (<a 
href="#XCerutti:06">Cerutti and Casas-Mollano</a>,&#x00A0;<a 
href="#XCerutti:06">2006</a>;&#x00A0;<a 
href="#XShabalina:08">Shabalina and
Koonin</a>,&#x00A0;<a 
href="#XShabalina:08">2008</a>) that leads to the suppression of double-stranded RNA molecules in a cell&#8217;s
cytoplasm.
</p><!--l. 183--><p class="indent" >  Throughout animals, canonical miRNAs are processed through a well-characterized
pathway. The primary precursor transcript (pri-miRNA) is transcribed by pol-II.
While in most cases the pri-miRNA is a long noncoding RNA, some miRNAs
are processed from protein-coding transcripts, where they are mostly derived
from introns (<a 
href="#XLin:06">Lin et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLin:06">2006</a>). In the next step, hairpin-shaped precursors, the
                                                               

                                                               
pre-miRNAs, are excised while the RNA is still residing in the nucleus. These are
exported into the cytoplasm (<a 
href="#XLund:04">Lund et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLund:04">2004</a>) and then processed further into
miRNA/miRNA* duplexes. In the final step the single-stranded mature miR or
its complement, the miR*, is incorporated into the RISC complex. Sequence
complementarity of miR and mRNA ensures the targeting specificity (<a 
href="#XBartel:13">Bartel</a>,&#x00A0;<a 
href="#XBartel:13">2009</a>).
As as a consequence, miRNAs share a set of structural characteristics, most
importantly the extremely stable secondary structure of the precursor hairpin
and the 2-bp overhang of miR and miR* generated by Dicer processing. These
features make it possible to reliably identify miRNAs from short RNA-seq data,
see e.g. (<a 
href="#XLangenberger:10a">Langenberger et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLangenberger:10a">2010</a>;&#x00A0;<a 
href="#XFriedlaender:12">Friedländer et&#x00A0;al.</a>,&#x00A0;<a 
href="#XFriedlaender:12">2012</a>;&#x00A0;<a 
href="#XLangenberger:13a">Langenberger
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLangenberger:13a">2012</a>).
</p><!--l. 200--><p class="indent" >  Most animal microRNAs are among the most highly conserved genetic elements. The
most stringent selection pressure acts on the mature miR sequence. This is a
consequence of the fact that a single miR typically targets a large number of
mRNAs. Mutations in the mature sequence thus simultaneously affect many
interactions, and thus are almost always selected against. In conjunction with
the stringent requirements on the secondary structure, the entire precursor is
under strong stabilizing selection (<a 
href="#XPrice:11">Price et&#x00A0;al.</a>,&#x00A0;<a 
href="#XPrice:11">2011</a>), explaining the observed
high levels of sequence conservation. As a consequence, even evolutionarily
distant homologs of miRNAs can be readily detected despite the short sequence
length. Most efficiently, <span 
class="pcrr7t-">infernal </span>(<a 
href="#XNawrocki:13">Nawrocki and Eddy</a>,&#x00A0;<a 
href="#XNawrocki:13">2013a</a>) is used for this
purpose, since it makes use of both sequence and structure comparison. The
evolution of miRNAs can thus be traced back in time with high accuracy (<a 
href="#XHertel:06a">Hertel
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHertel:06a">2006</a>).
</p><!--l. 214--><p class="indent" >  Like other gene families, miRNAs form paralogs (<a 
href="#XTanzer:04a">Tanzer and Stadler</a>,&#x00A0;<a 
href="#XTanzer:04a">2004</a>;&#x00A0;<a 
href="#XHertel:12a">Hertel
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHertel:12a">2012</a>) and hence often appear as families of homologous genes. This forms the
basis of the <span 
class="pcrr7t-">miRBase </span>nomenclature (<a 
href="#XAmbrose:03a">Ambros et&#x00A0;al.</a>,&#x00A0;<a 
href="#XAmbrose:03a">2003</a>). A series of investigations
into the phylogenetic distribution of miRNA families showed that miRNAs are
infrequently lost at family level and thus serve as excellent phylogenetic markers
(<a 
href="#XSempere:06">Sempere et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSempere:06">2006</a>;&#x00A0;<a 
href="#XHeimberg:08">Heimberg et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHeimberg:08">2007</a>,&#x00A0;<a 
href="#XHeimberg:10">2010</a>;&#x00A0;<a 
href="#XWheeler:09">Wheeler et&#x00A0;al.</a>,&#x00A0;<a 
href="#XWheeler:09">2009</a>), although
the massive restructuring of the miRNA complement of tunicates is an important
exception to this rule (<a 
href="#XFu:08">Fu et&#x00A0;al.</a>,&#x00A0;<a 
href="#XFu:08">2008</a>).
</p><!--l. 223--><p class="indent" >  The innovation of new miRNA families is an on-going process. Experimental surveys
of the miRNA repertoir thus have reported a large number of very young and even
species-specific miRNAs (<a 
href="#XBentwich:05">Bentwich et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBentwich:05">2005</a>;&#x00A0;<a 
href="#XBerezikov:06">Berezikov et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBerezikov:06">2006</a>). The process
was studied quantitatively in fruit flies, where innovation rate of as many as <span 
class="zptmcm7t-">12 </span>new
miRNA genes per million years has been estimated (<a 
href="#XLu:08">Lu et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLu:08">2008</a>). This is consistent
with the fact that stable hairpins are abundant structural elements in random RNAs, which
makes it not only possible but actually quite likely that miRNA precursors appear by
chance in transcribed genomic regions (<a 
href="#XTanzer:04a">Tanzer and Stadler</a>,&#x00A0;<a 
href="#XTanzer:04a">2004</a>;&#x00A0;<a 
href="#XCampoPaysaa:11">Campo-Paysaa
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XCampoPaysaa:11">2011a</a>;&#x00A0;<a 
href="#XMarco:13">Marco et&#x00A0;al.</a>,&#x00A0;<a 
href="#XMarco:13">2013</a>). Of course, only a tiny fraction of these fortuitously
processed hairpins have a function and hence are subject to selection, and an even smaller
subset is conserved over long evolutionary time scales. Detailed studies showed that
evolutionarily young miRNA have comparably low expression levels. Initially, they go
through a phase of relatively fast sequence evolution (<a 
href="#XLiang:09">Liang and Li</a>,&#x00A0;<a 
href="#XLiang:09">2009</a>;&#x00A0;<a 
href="#XMeunier:12">Meunier
                                                               

                                                               
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XMeunier:12">2012</a>), which slows down as the selective pressures from a gradual increase in the
number of target site increases. A large, diverse set of targets then protects against
miRNA loss (<a 
href="#XLee:07a">Lee et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLee:07a">2007</a>). The rate of gain of miRNA families that are
retained essentially permanently amounts to only <span 
class="zptmcm7t-">1 </span>per several million years. This
number is consistent with divergence of the miRNA complements between animal
phyla.
</p><!--l. 245--><p class="indent" >  Many authors have observed that overall the miRNA repertoire has been expanding
throughout animal evolution in a manner that at least roughly correlates with
morphological complexity (<a 
href="#XHertel:06a">Hertel et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHertel:06a">2006</a>;&#x00A0;<a 
href="#XSempere:06">Sempere et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSempere:06">2006</a>;&#x00A0;<a 
href="#XNiwa:07">Niwa and
Slack</a>,&#x00A0;<a 
href="#XNiwa:07">2007</a>;&#x00A0;<a 
href="#XProchnik:07">Prochnik et&#x00A0;al.</a>,&#x00A0;<a 
href="#XProchnik:07">2007</a>;&#x00A0;<a 
href="#XLee:07a">Lee et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLee:07a">2007</a>;&#x00A0;<a 
href="#XHeimberg:08">Heimberg et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHeimberg:08">2007</a>;&#x00A0;<a 
href="#XPeterson:09">Peterson
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XPeterson:09">2009</a>;&#x00A0;<a 
href="#XBerezikov:11">Berezikov</a>,&#x00A0;<a 
href="#XBerezikov:11">2011</a>). Several bursts of miRNA innovation have been observed
(<a 
href="#XHertel:06a">Hertel et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHertel:06a">2006</a>;&#x00A0;<a 
href="#XHeimberg:08">Heimberg et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHeimberg:08">2007</a>;&#x00A0;<a 
href="#XTanzer:10a">Tanzer et&#x00A0;al.</a>,&#x00A0;<a 
href="#XTanzer:10a">2010</a>;&#x00A0;<a 
href="#XHertel:15a">Hertel and
Stadler</a>,&#x00A0;<a 
href="#XHertel:15a">2015</a>), most notably at the root of the placental mammals, the ancestor of
&#8220;free-living&#8221; nematodes, or the radiation of the drosophilids. Massive morphological
simplification, on the other hand, is sometimes associated with a drastic loss of miRNA
families. This has been observed most prominently for tunicates (<a 
href="#XFu:08">Fu et&#x00A0;al.</a>,&#x00A0;<a 
href="#XFu:08">2008</a>;&#x00A0;<a 
href="#XDai:09">Dai
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XDai:09">2009</a>).
</p><!--l. 257--><p class="noindent" >
</p>
  <h4 class="subsectionHead"><span class="titlemark">2.2  </span> <a 
 id="x1-50002.2"></a>miRNA identification and validation</h4>
<!--l. 259--><p class="noindent" >The first miRNA reported in any tunicate was let-7, which was first detected in <span 
class="ptmri7t-">Ciona</span>
<span 
class="ptmri7t-">robusta </span>and <span 
class="ptmri7t-">Herdmania curvata </span>(<a 
href="#XPasquinelli2000">Pasquinelli et&#x00A0;al.</a>,&#x00A0;<a 
href="#XPasquinelli2000">2000</a>). A previous study the same
year in <span 
class="ptmri7t-">C.</span><span 
class="ptmri7t-">&#x00A0;elegans </span>had shown that small RNA let-7 (<span 
class="zptmcm7t-">21 </span>nt) was required for late larval to
adult developmental transition (<a 
href="#XReinhart:2000mz">Reinhart et&#x00A0;al.</a>,&#x00A0;<a 
href="#XReinhart:2000mz">2000</a>). Small RNA let-7 was then shown
to also be differentially expressed during the development of many distantly related
animal taxa, but was not detected in Porifera, Ctenophora, Cnidaria, and Acoelomorpha,
suggesting that let-7 was involved in the regulation of late temporal transitions during
development or in the evolution of complex life histories in the Nephrozoa (<a 
href="#XPasquinelli2000">Pasquinelli
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XPasquinelli2000">2000</a>,&#x00A0;<a 
href="#XPasquinelli2003">2003</a>).
</p><!--l. 270--><p class="indent" >  The first systematic computational screen of a tunicate genome in 2005 followed the
first large animal sequencing projects. Beginning with <span 
class="ptmri7t-">C.</span><span 
class="ptmri7t-">&#x00A0;robusta </span>and <span 
class="ptmri7t-">C.</span><span 
class="ptmri7t-">&#x00A0;savignyi</span>, a
profile-based strategy was implemented in the ERPIN program (<a 
href="#XLegendre2005">Legendre et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLegendre2005">2005</a>).
This work detected a set of new miRNAs candidates considered to be <span 
class="ptmri7t-">C.</span><span 
class="ptmri7t-">&#x00A0;robusta </span>specific,
such as the members of the family miR-9 and miR-79 together with many other miRNA
families that, as expected, were found to be homologous between both Ciona species.
Among these were miR-124, miR-92, miR-98, miR-325, the miR310-313 group, and
let-7. In the same year, a whole-genomic comparative approach in the urochordate
lineage was performed on the species <span 
class="ptmri7t-">C.</span><span 
class="ptmri7t-">&#x00A0;robusta</span>, <span 
class="ptmri7t-">C.</span><span 
class="ptmri7t-">&#x00A0;savignyi</span>, and <span 
class="ptmri7t-">O.</span><span 
class="ptmri7t-">&#x00A0;dioica</span>.
Using a computational screening of structured ncRNAs based upon homology
between predicted precursor hairpin structures <span 
class="zptmcm7t-">41 </span>miRNA candidates were detected
                                                               

                                                               
including let-7 and other six known candidates in <span 
class="ptmri7t-">C.</span><span 
class="ptmri7t-">&#x00A0;robusta </span>(<a 
href="#XMissal2005">Missal et&#x00A0;al.</a>,&#x00A0;<a 
href="#XMissal2005">2005</a>).
The same group in 2007 implemented a structure-based clustering approach in
<span 
class="ptmri7t-">C. robusta </span>predicted the presence of <span 
class="zptmcm7t-">58 </span>miRNAs, of which only let-7, miR-7,
miR-124, and miR-126 coincided with the previously annotated miRNAs (<a 
href="#XWill2007">Will
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XWill2007">2007</a>).
</p><!--l. 289--><p class="indent" >  Up to this point, efforts to map the miRNAs in urochordate lineages were mainly
focused on computational approaches. With improved sequence technologies, new hybrid
strategies were employed that combined computational and experimental studies to
validate candidate families previously detected. For instance, the first bona fide records for
<span 
class="ptmri7t-">C.</span><span 
class="ptmri7t-">&#x00A0;robusta </span>miRNAs were registered in mirBase only in Release 11 based on the data
reported in <a 
href="#XNorden-Krichmar2007">Norden-Krichmar et&#x00A0;al.</a>&#x00A0;(<a 
href="#XNorden-Krichmar2007">2007</a>). This study searched the genomes of
<span 
class="ptmri7t-">C.</span><span 
class="ptmri7t-">&#x00A0;robusta </span>and <span 
class="ptmri7t-">C.</span><span 
class="ptmri7t-">&#x00A0;savignyi </span>for conservation of the seed regions of the known
mature miRNA sequences compiled in the 2006 release of miRBase release. Those
miRNAs were aligned locally using the FASTA/ssearch34 program, retaining only
matches with at least 90% identity. The characteristic hairpin structure and the
relative positioning of the mature sequences were used as additional filters. After
manual curation a set of <span 
class="zptmcm7t-">18 </span>miRNAs that appeared conserved in both Cionas was
reported. The expression of <span 
class="zptmcm7t-">14 </span>of these families, including let-7, miR-7, and
miR-126, was then confirmed by Northern blot analyses in the adult tissue of
<span 
class="ptmri7t-">C.</span><span 
class="ptmri7t-">&#x00A0;robusta</span>.
</p><!--l. 307--><p class="indent" >  Until 2008, most of the miRNAs were annotated in Cionas, and few new miRNA
families were discovered in other urochordates. The first repertory of miRNAs in tunicate
beyond the Ciona species was published in 2008 for the larvacean <span 
class="ptmri7t-">O.</span><span 
class="ptmri7t-">&#x00A0;dioica </span>(<a 
href="#XFu:08">Fu
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XFu:08">2008</a>). At that time the Fu <span 
class="ptmri7t-">et al.</span>&#x00A0;were studying the temporal-spatial expression
patterns of conserved miRNAs in different developmental stages of oocytes, 1-cell zygote,
2-8 cell embryons, blastulas, gastrulas, tadpoles (in different stages) and 1-6 days old
adults from <span 
class="ptmri7t-">O.</span><span 
class="ptmri7t-">&#x00A0;dioica</span>. Small RNAs were isolated, amplified by RT-PCR and rapid
amplification of cDNA ends (RACE) of the developmental stages, cloned and sequenced.
Blast searches using the sequences of cloned small RNA libraries were used
to annotate small RNAs as miRNA candidates. In further steps the recovered
genomic flanking sequences each side of those mapped candidates were used as
input to predicted secondary structures by mfold v3.1.&#x00A0;This step was used to
detect candidates that folds like miRNA hairpins and aimed to decrease the set of
false positive potential miRNAs in <span 
class="ptmri7t-">O.</span><span 
class="ptmri7t-">&#x00A0;dioica</span>. Finally, for this set of potential
candidates a developmental miRNA array dot blot analyses were performed to
detect miRNA expression. With this approach from <span 
class="zptmcm7t-">3066 </span>sequenced small RNA
clones, expression was detected for only <span 
class="zptmcm7t-">55 </span>miRNAs. The authors suggested that
these candidates were expressed throughout the short life cycle of <span 
class="ptmri7t-">O.</span><span 
class="ptmri7t-">&#x00A0;dioica</span>
showing that some of them were stocked as maternal determinants prior to rapid
embryonic development. The authors also identified a set of sex-specific miRNAs
that appeared as male/female gonad differentiation became apparent and was
maintained throughout spermatogenesis (<a 
href="#XFu:08">Fu et&#x00A0;al.</a>,&#x00A0;<a 
href="#XFu:08">2008</a>). Unexpectedly, the majority
of the miRNAs loci in <span 
class="ptmri7t-">O.</span><span 
class="ptmri7t-">&#x00A0;dioica </span>were located in antisense orientations in the
host genes, in contrast to the majority of the mammalian miRNAs known at the
time.
                                                               

                                                               
</p><!--l. 337--><p class="indent" >  Between the years 2009 and 2015 the majority of the studies of miRNAs in tunicates
were focused on the validation of expression of computational predicted miRNAs in
Cionas with the special focus on <span 
class="ptmri7t-">C.</span><span 
class="ptmri7t-">&#x00A0;robusta </span>as model organism of tunicates
or on the test of new computational approaches as miRTRAP, miRDeep2 and
miRRim2, which used next-generation sequencing libraries of small RNAs derived
from <span 
class="ptmri7t-">C. robusta </span>to validate their algorithms. Then by the year 2016 the first
comparative homology based search strategy let us to identify the repertory on
miRNAs and other ncRNAs in the carpet sea squirt <span 
class="ptmri7t-">Didemnum vexillum </span>with a
preliminary comparative analysis of gain and losses of miRNA families in chordates
which included the <span 
class="ptmri7t-">Cionas</span>, <span 
class="ptmri7t-">O. dioca </span>and the colonial tunicate <span 
class="ptmri7t-">Botryllus schlosseri</span>
(<a 
href="#XVelandia-Huerto2016">Velandia-Huerto et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVelandia-Huerto2016">2016</a>). By the same year, from the preliminary genome
sequence assembled for the Southern Ocean salp, <span 
class="ptmri7t-">Salpa thompsoni </span>(Urochordata,
Thaliacea) a set of miRNAs families were detected (<a 
href="#XJue2016">Jue et&#x00A0;al.</a>,&#x00A0;<a 
href="#XJue2016">2016</a>) and in 2017 the
prediction of miRNAs families were reported for <span 
class="ptmri7t-">Halocynthia roretzi </span>species (<a 
href="#XWang2017">Wang
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XWang2017">2017</a>).
</p><!--l. 355--><p class="noindent" >
</p>
  <h5 class="subsubsectionHead"><span class="titlemark">2.2.1  </span> <a 
 id="x1-60002.2.1"></a>High throughput studies of Ciona miRNAs</h5>
<!--l. 357--><p class="noindent" >At the end of the last decade the application of next generation sequencing technologies to
sequence small RNA libraries changed the common way used to detect expression of
miRNAs in many organisms including the tunicates. This technology became one of the
most common approaches that supported methods like RT-PCR, microarrays or dot
blotting which were previously used to validate miRNA expression in tunicates. In 2009
small RNA libraries prepared from various developmental stages including unfertilized
eggs, early embryos, late embryos and adults from <span 
class="ptmri7t-">C.</span><span 
class="ptmri7t-">&#x00A0;robusta </span>were assayed by
high-throughput sequencing of cDNA with an Illumina 1G Genome Analyzer. These
sequencing data revealed <span 
class="zptmcm7t-">80 </span>miRNAs families in <span 
class="ptmri7t-">C.</span><span 
class="ptmri7t-">&#x00A0;robusta</span>. Unexpectedly, a distinct
species of small RNAs processed from the distal parts of the miRNA precursor hairpins
was found to be abundantly expressed. These were termed miRNA-offset RNAs (moRs)
(<a 
href="#XShi2009">Shi et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShi2009">2009</a>). Later on, after extracting non-coding conserved regions of
whole genome alignments between <span 
class="ptmri7t-">C.</span><span 
class="ptmri7t-">&#x00A0;robusta </span>and <span 
class="ptmri7t-">C.</span><span 
class="ptmri7t-">&#x00A0;savigny </span>a set of <span 
class="zptmcm7t-">12 </span>million
sequences were computationally folded using RNAfold and mfold. Then after
combining the following criteria: structure/sequence conservation, homology to
known miRNAs, and phylogenetic footprinting the authors detected a set of <span 
class="zptmcm7t-">458</span>
candidate sequences (<a 
href="#XKeshavan2010">Keshavan et&#x00A0;al.</a>,&#x00A0;<a 
href="#XKeshavan2010">2010</a>). Then in order to validate those
candidates, RT-PCR and PAGE were conducted to design a custom microarray. <span 
class="zptmcm7t-">244 </span>of
the <span 
class="zptmcm7t-">458 </span>miRNA predictions were represented either in their microarray data
or in the Illumina sequences from (<a 
href="#XShi2009">Shi et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShi2009">2009</a>), leading to an estimate
of about <span 
class="zptmcm7t-">300 </span>miRNA genes in <span 
class="ptmri7t-">C.</span><span 
class="ptmri7t-">&#x00A0;robusta</span>. It is worth noting that (<a 
href="#XKeshavan2010">Keshavan
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XKeshavan2010">2010</a>) did not recover <span 
class="zptmcm7t-">39 </span>previously characterized miRNAs. A novel
computational strategy for the systematic, whole-genome identification of microRNA
                                                               

                                                               
from high throughput sequencing information was developed in 2010 in <a 
href="#XHendrix2010">Hendrix
et&#x00A0;al.</a>&#x00A0;(<a 
href="#XHendrix2010">2010</a>) and applied to <span 
class="ptmri7t-">C.</span><span 
class="ptmri7t-">&#x00A0;robusta</span>. This method, miRTRAP, relies on the sequence
patterns produced by mechanisms of microRNA biogenesis but also includes
additional criteria regarding the prevalence and quality of small RNAs arising from
the antisense strand and the neighboring loci. With that approach, nearly <span 
class="zptmcm7t-">400</span>
putative microRNAs loci were detected. The miRTRAP approach depends crucially
on the depth of the small RNAs mapped to a given locus and requires a very
accurate assignment of small RNA sequences on their relative positions along the
hairpin, that is, miR/miR*, moR/moR* and loop (<a 
href="#XHendrix2010">Hendrix et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHendrix2010">2010</a>). A further
improvement in the analysis of miRNA sequencing data became available with
miRDeep2, a revised version miRDeep (<a 
href="#XFriedlaender:12">Friedländer et&#x00A0;al.</a>,&#x00A0;<a 
href="#XFriedlaender:12">2012</a>). It was reported to
identify with an accuracy of <span 
class="zptmcm7t-">98</span><span 
class="zptmcm7m-">.</span><span 
class="zptmcm7t-">6</span>% and <span 
class="zptmcm7t-">99</span><span 
class="zptmcm7m-">.</span><span 
class="zptmcm7t-">9</span>% canonical and non-canonical
miRNAs in different species. A reanalysis of the small RNA data from (<a 
href="#XShi2009">Shi
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShi2009">2009</a>) with miRDeep2 reported <span 
class="zptmcm7t-">313 </span>known and <span 
class="zptmcm7t-">127 </span>novel miRNAs in
<span 
class="ptmri7t-">C.</span><span 
class="ptmri7t-">&#x00A0;robusta</span>. In the same year the program miRRim2 (<a 
href="#XTerai2012">Terai et&#x00A0;al.</a>,&#x00A0;<a 
href="#XTerai2012">2012</a>) was
applied to the <span 
class="ptmri7t-">C.</span><span 
class="ptmri7t-">&#x00A0;robusta </span>genome, in which some candidates identified from the
work of <a 
href="#XHendrix2010">Hendrix et&#x00A0;al.</a>&#x00A0;(<a 
href="#XHendrix2010">2010</a>) and the several novel promising candidates were
detected.
</p><!--l. 404--><p class="indent" >  In 2013, (<a 
href="#XKusakabe2013">Kusakabe et&#x00A0;al.</a>,&#x00A0;<a 
href="#XKusakabe2013">2013</a>) investigated the expression patterns of the cluster
miR-1 and miR-133 in <span 
class="ptmri7t-">C.</span><span 
class="ptmri7t-">&#x00A0;robusta </span>and in <span 
class="ptmri7t-">C.</span><span 
class="ptmri7t-">&#x00A0;savignyi</span>. RT-PCR amplification of
miR-1/133 precursors were performed and PCR products were subcloned and sequenced.
Whole-mount in situ hybridization to detect cin-miR-1/miR-133 primary transcript was
performed and LNA Northern blotting was conducted on different developmental
stages.
</p><!--l. 411--><p class="noindent" >
</p>
  <h5 class="subsubsectionHead"><span class="titlemark">2.2.2  </span> <a 
 id="x1-70002.2.2"></a>High throughput miRNA searches in other Urochordates</h5>
<!--l. 413--><p class="noindent" >Since 2016 new approximations has increased our knowledge about new families in other
tunicates thanks to the sequence of new urochordate genomes of the species
<span 
class="ptmri7t-">D.</span><span 
class="ptmri7t-">&#x00A0;vexillum</span>, <span 
class="ptmri7t-">S.</span><span 
class="ptmri7t-">&#x00A0;thompsoni </span>and <span 
class="ptmri7t-">H.</span><span 
class="ptmri7t-">&#x00A0;roretzi</span>. A detailed homology-based computational
survey of ncRNAs was performend for an early draft genome of <span 
class="ptmri7t-">D. vexillum</span>
(<a 
href="#XVelandia-Huerto2016">Velandia-Huerto et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVelandia-Huerto2016">2016</a>). Blast and HMMer searches were performed with
annotated small ncRNAs sequences from metazoans and hidden markov models from
RFAM<span class="footnote-mark"><a 
href="tunicates_ncrna3.html#fn2x0"><sup class="textsuperscript">2</sup></a></span><a 
 id="x1-7001f2"></a> 
to obtain the sort of candidates at sequence level. Structural alignments of those sequences
were performed by infernal (<a 
href="#XNawrocki:2013">Nawrocki and Eddy</a>,&#x00A0;<a 
href="#XNawrocki:2013">2013b</a>), using metazoan-specific
covariance models to annotate the small ncRNAs collection, which <span 
class="zptmcm7t-">57 </span>families and <span 
class="zptmcm7t-">100</span>
loci of miRNAs were found.
                                                               

                                                               
</p><!--l. 426--><p class="indent" >  Small RNA libraries for the Southern Ocean salp <span 
class="ptmri7t-">S.</span><span 
class="ptmri7t-">&#x00A0;thompsoni </span>were sequenced with an
Illumina Hiseq 2000 (<a 
href="#XJue2016">Jue et&#x00A0;al.</a>,&#x00A0;<a 
href="#XJue2016">2016</a>). After filtering data sets to 18-24 nt for miRNA
and 28-32 nt for piRNA, the reads were aligned to <span 
class="ptmri7t-">S. thompsoni </span>genome and miRNA gene
folding predictions were performed using RNAfold. In this initial survey of small
RNAs, were revealed the presence of known, conserved miRNAs, as well as
novel miRNA genes and mature miRNA signatures for varying developmental
stages. Then in 2017, the prediction of <span 
class="zptmcm7t-">319 </span>miRNAs candidates in <span 
class="ptmri7t-">H.</span><span 
class="ptmri7t-">&#x00A0;roretzi </span>were
obtained through three complementary methods. The experimental validation
suggested that more than half of these candidate miRNAs are expressed during
embryogenesis. The expression of some of the predicted miRNAs were validated by
RT-PCR using embryonic RNA. In this approach <span 
class="ptmri7t-">C.</span><span 
class="ptmri7t-">&#x00A0;robusta </span>small RNA-Seq reads
derived from <span 
class="ptmri7t-">C.</span><span 
class="ptmri7t-">&#x00A0;robusta </span>(<a 
href="#XShi2009">Shi et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShi2009">2009</a>) (previously known as <span 
class="ptmri7t-">C. intestinalis</span>
today reclassified) was used to identify conserved miRNAs in <span 
class="ptmri7t-">H.</span><span 
class="ptmri7t-">&#x00A0;roretzi </span>(<a 
href="#XWang2017">Wang
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XWang2017">2017</a>).
</p><!--l. 443--><p class="indent" >  Current repertoire of miRNAs (Figure <a 
href="#x1-80198">8<!--tex4ht:ref: fig:matrimirnas --></a>) is based on the final matrix of miRNAs
families from <a 
href="#XHertel:15a">Hertel and Stadler</a>&#x00A0;(<a 
href="#XHertel:15a">2015</a>) and complemented by homology methods
developed on <a 
href="#XVelandia-Huerto2016">Velandia-Huerto et&#x00A0;al.</a>&#x00A0;(<a 
href="#XVelandia-Huerto2016">2016</a>) and specifically for <span 
class="ptmri7t-">S. thompsoni </span>and <span 
class="ptmri7t-">H.</span>
<span 
class="ptmri7t-">roretzi </span>blast searches with structural alignments where applied on the reported candidates
in <a 
href="#XJue2016">Jue et&#x00A0;al.</a>&#x00A0;(<a 
href="#XJue2016">2016</a>) and <a 
href="#XWang2017">Wang et&#x00A0;al.</a>&#x00A0;(<a 
href="#XWang2017">2017</a>).
</p>
  <h4 class="subsectionHead"><span class="titlemark">2.3  </span> <a 
 id="x1-80002.3"></a>miRNA in clusters</h4>
<!--l. 454--><p class="noindent" >One of the most interesting aspects about the patterns of genomic locations of
miRNAs is to know whether those loci are randomly distributed throughout the
genome as single copies or if they are arranged on consecutive locations or in
tandem copies clustered to be expressed from polycistronic primary precursors
or to be transcribed independently (<a 
href="#XTanzer:10a">Tanzer et&#x00A0;al.</a>,&#x00A0;<a 
href="#XTanzer:10a">2010</a>). Interestingly in <span 
class="ptmri7t-">O.</span>
<span 
class="ptmri7t-">dioca </span>miRNAs are located in the antisense orientations of protein-coding gene
and immediately downstream of its corresponding 3&#8216;UTR region or even more
frequently in the sense strand of introns (<a 
href="#XFu:08">Fu et&#x00A0;al.</a>,&#x00A0;<a 
href="#XFu:08">2008</a>). Nevertheless, after those
conspicuous distributions some clusters have been also identified in <span 
class="ptmri7t-">O. dioca</span>. For
instance four miRNAs, miR-1490a, miR-1493, miR-1497d, and miR-1504, are
reported by <a 
href="#XFu:08">Fu et&#x00A0;al.</a>&#x00A0;(<a 
href="#XFu:08">2008</a>) to be present as two copies, and miR-1497d-1 and
miR-1497d-2 are included in the large miR-1497 cluster. The current structure of this
cluster is shown in Table <a 
href="#x1-8001r1">1<!--tex4ht:ref: tab:detailBigClusters --></a> although only one copy for the miR-1497 has been
reported for <span 
class="ptmri7t-">C. robusta </span>located in an intergenic region (<a 
href="#XFu:08">Fu et&#x00A0;al.</a>,&#x00A0;<a 
href="#XFu:08">2008</a>;&#x00A0;<a 
href="#XHendrix2010">Hendrix
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHendrix2010">2010</a>) and one in <span 
class="ptmri7t-">C. savigny </span>overlapped in an intron (<a 
href="#XFu:08">Fu et&#x00A0;al.</a>,&#x00A0;<a 
href="#XFu:08">2008</a>). By testing
real time PCR co-expression of some miRNAs, their host and adjacent genes in
<span 
class="ptmri7t-">O. dioca </span>by <a 
href="#XFu:08">Fu et&#x00A0;al.</a>&#x00A0;(<a 
href="#XFu:08">2008</a>) it was discovered that in the case of the cluster
miR-1487/miR-1488 there was no clear positive or negative correlation with the
expression of its anti-sense hosting gene. The same authors showed that in males this
cluster expression was not associated with the expression of its adjacent ABCA3
                                                               

                                                               
gene.
</p>
<div class="center" 
>
<!--l. 480--><p class="noindent" >
<a 
 id="x1-8001r1"></a>
</p><!--l. 482--><div class="longtable"> <table id="TBL-1" class="longtable" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-1-1g"><col 
id="TBL-1-1" /><col 
id="TBL-1-2" /><col 
id="TBL-1-3" /><col 
id="TBL-1-4" /><col 
id="TBL-1-5" /><col 
id="TBL-1-6" /><col 
id="TBL-1-7" /></colgroup>
<tr  
 style="vertical-align:baseline;" id="TBL-1-1-"><td colspan="3" style="white-space:wrap; text-align:left;" id="TBL-1-1-1"  
class="td11"></td>             <div class="multicolumn"  style="white-space:nowrap; text-align:center;"><div class="caption" 
><span class="id"><span 
class="ptmr7t-x-x-85">Table</span><span 
class="ptmr7t-x-x-85">&#x00A0;1: </span></span><span  
class="content"><span 
class="ptmr7t-x-x-85">Details of biggest miRNA cluster for chordate species.</span></span></div><!--tex4ht:label?: x1-8001r1 -->                                 </div>                                <a 
 id="x1-8002"></a>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-1-2-"><td  style="white-space:wrap; text-align:left;" id="TBL-1-2-1"  
class="td11">           </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-3-"><td  style="white-space:wrap; text-align:left;" id="TBL-1-3-1"  
class="td11">
<!--l. 486--><p class="noindent" >           </p></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-4-"><td  style="white-space:wrap; text-align:left;" id="TBL-1-4-1"  
class="td11">
<!--l. 487--><p class="noindent" ><span 
class="ptmb7t-x-x-85">Specie</span>                </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-4-2"  
class="td11">
<!--l. 487--><p class="noindent" ><span 
class="ptmb7t-x-x-85">Chr</span>                    </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-4-3"  
class="td11">
<!--l. 487--><p class="noindent" ><span 
class="ptmb7t-x-x-85">Start</span>                  </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-4-4"  
class="td11">
<!--l. 487--><p class="noindent" ><span 
class="ptmb7t-x-x-85">End</span>                    </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-4-5"  
class="td11">
<!--l. 488--><p class="noindent" ><span 
class="ptmb7t-x-x-85">Size</span>
<span 
class="ptmb7t-x-x-85">(Mb)</span>       </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-4-6"  
class="td11">
<!--l. 488--><p class="noindent" ><span 
class="ptmb7t-x-x-85">No.</span>        </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-4-7"  
class="td11">
<!--l. 488--><p class="noindent" ><span 
class="ptmb7t-x-x-85">miRNAs detail</span>          </p></td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-1-5-"><td  style="white-space:wrap; text-align:left;" id="TBL-1-5-1"  
class="td11">           </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-6-"><td  style="white-space:wrap; text-align:left;" id="TBL-1-6-1"  
class="td11">
<!--l. 489--><p class="noindent" >           </p></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-7-"><td  style="white-space:wrap; text-align:left;" id="TBL-1-7-1"  
class="td11">
<!--l. 491--><p class="noindent" ><span 
class="ptmri7t-x-x-85">B. floridae</span>          </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-7-2"  
class="td11">
<!--l. 491--><p class="noindent" ><span 
class="ptmr7t-x-x-85">Bf</span><span 
class="ptmr7t-x-x-85">_V2</span><span 
class="ptmr7t-x-x-85">_118</span>        </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-7-3"  
class="td11">
<!--l. 491--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">216744</span>           </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-7-4"  
class="td11">
<!--l. 491--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">220351</span>           </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-7-5"  
class="td11">
<!--l. 491--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">3607</span>      </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-7-6"  
class="td11">
<!--l. 491--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">5</span>         </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-7-7"  
class="td11">
<!--l. 492--><p class="noindent" ><span 
class="ptmr7t-x-x-85">bfl-mir-4869,</span>
<span 
class="ptmr7t-x-x-85">bfl-mir-4857,</span>
<span 
class="ptmr7t-x-x-85">bfl-mir-4862,</span>
<span 
class="ptmr7t-x-x-85">bfl-mir-4856b,</span>
<span 
class="ptmr7t-x-x-85">bfl-mir-4856a</span>             </p></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-8-"><td  style="white-space:wrap; text-align:left;" id="TBL-1-8-1"  
class="td11">
<!--l. 493--><p class="noindent" ><span 
class="ptmri7t-x-x-85">O. dioica</span>            </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-8-2"  
class="td11">
<!--l. 493--><p class="noindent" ><span 
class="ptmr7t-x-x-85">scaffold</span><span 
class="ptmr7t-x-x-85">_3</span>          </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-8-3"  
class="td11">
<!--l. 493--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">2222857</span>          </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-8-4"  
class="td11">
<!--l. 493--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">2223714</span>          </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-8-5"  
class="td11">
<!--l. 493--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">857</span>        </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-8-6"  
class="td11">
<!--l. 493--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">6</span>         </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-8-7"  
class="td11">
<!--l. 494--><p class="noindent" ><span 
class="ptmr7t-x-x-85">odi-mir-1497e,</span>
<span 
class="ptmr7t-x-x-85">odi-mir-1497d-2,</span>
<span 
class="ptmr7t-x-x-85">odi-mir-1497d-1,</span>
<span 
class="ptmr7t-x-x-85">odi-mir-1497c,</span>
<span 
class="ptmr7t-x-x-85">odi-mir-1497b,</span>
<span 
class="ptmr7t-x-x-85">odi-mir-1497a</span>            </p></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-9-"><td  style="white-space:wrap; text-align:left;" id="TBL-1-9-1"  
class="td11">
<!--l. 496--><p class="noindent" ><span 
class="ptmri7t-x-x-85">B. schlosseri</span>      </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-9-2"  
class="td11">
<!--l. 496--><p class="noindent" ><span 
class="ptmr7t-x-x-85">chrUn</span>                 </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-9-3"  
class="td11">
<!--l. 496--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">40003</span>             </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-9-4"  
class="td11">
<!--l. 496--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">41320</span>             </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-9-5"  
class="td11">
<!--l. 496--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">1317</span>      </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-9-6"  
class="td11">
<!--l. 496--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">2</span>         </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-9-7"  
class="td11">
<!--l. 496--><p class="noindent" ><span 
class="ptmr7t-x-x-85">mir-233, mir-10</span>         </p></td>
                                                               

                                                               
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-10-"><td  style="white-space:wrap; text-align:left;" id="TBL-1-10-1"  
class="td11">
<!--l. 498--><p class="noindent" ><span 
class="ptmri7t-x-x-85">C. robusta</span>          </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-10-2"  
class="td11">
<!--l. 498--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">7</span>                   </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-10-3"  
class="td11">
<!--l. 498--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">4153284</span>          </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-10-4"  
class="td11">
<!--l. 498--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">4156782</span>          </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-10-5"  
class="td11">
<!--l. 498--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">3498</span>      </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-10-6"  
class="td11">
<!--l. 498--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">23</span>       </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-10-7"  
class="td11">
<!--l. 499--><p class="noindent" ><span 
class="ptmr7t-x-x-85">cin-mir-4006d,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4006c,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4001b-2,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4000i,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4006g,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4001e,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4001d,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4000g,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4006f,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4006b,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4001b-1,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4000c,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4006e,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4000b-2,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4001a-1,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4000b-1,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4002,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4000d,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4001h,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4000a-2,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4006a-2,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4006a-3,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4006a-1</span>         </p></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-11-"><td  style="white-space:wrap; text-align:left;" id="TBL-1-11-1"  
class="td11">
<!--l. 504--><p class="noindent" ><span 
class="ptmri7t-x-x-85">C. savignyi</span>         </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-11-2"  
class="td11">
<!--l. 504--><p class="noindent" ><span 
class="ptmr7t-x-x-85">reftig</span><span 
class="ptmr7t-x-x-85">_16</span>            </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-11-3"  
class="td11">
<!--l. 504--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">3924783</span>          </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-11-4"  
class="td11">
<!--l. 504--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">3925336</span>          </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-11-5"  
class="td11">
<!--l. 504--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">553</span>        </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-11-6"  
class="td11">
<!--l. 504--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">3</span>         </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-11-7"  
class="td11">
<!--l. 505--><p class="noindent" ><span 
class="ptmr7t-x-x-85">csa-mir-216b,</span>
<span 
class="ptmr7t-x-x-85">csa-mir-216a,</span>
<span 
class="ptmr7t-x-x-85">csa-mir-217</span>                </p></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-12-"><td  style="white-space:wrap; text-align:left;" id="TBL-1-12-1"  
class="td11">
<!--l. 506--><p class="noindent" ><span 
class="ptmri7t-x-x-85">C. savignyi</span>         </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-12-2"  
class="td11">
<!--l. 506--><p class="noindent" ><span 
class="ptmr7t-x-x-85">reftig</span><span 
class="ptmr7t-x-x-85">_1</span>              </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-12-3"  
class="td11">
<!--l. 506--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">1335375</span>          </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-12-4"  
class="td11">
<!--l. 506--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">1336487</span>          </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-12-5"  
class="td11">
<!--l. 506--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">1112</span>      </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-12-6"  
class="td11">
<!--l. 506--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">3</span>         </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-12-7"  
class="td11">
<!--l. 507--><p class="noindent" ><span 
class="ptmr7t-x-x-85">csa-mir-92b,</span>
<span 
class="ptmr7t-x-x-85">csa-mir-92c,</span>
<span 
class="ptmr7t-x-x-85">csa-mir-92a</span>                </p></td>
                                                               

                                                               
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-13-"><td  style="white-space:wrap; text-align:left;" id="TBL-1-13-1"  
class="td11">
<!--l. 508--><p class="noindent" ><span 
class="ptmri7t-x-x-85">D. rerio</span>              </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-13-2"  
class="td11">
<!--l. 508--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">4</span>                   </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-13-3"  
class="td11">
<!--l. 508--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">28738556</span>        </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-13-4"  
class="td11">
<!--l. 508--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">28754891</span>        </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-13-5"  
class="td11">
<!--l. 508--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">16335</span>     </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-13-6"  
class="td11">
<!--l. 508--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">60</span>       </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-13-7"  
class="td11">
<!--l. 509--><p class="noindent" ><span 
class="ptmr7t-x-x-85">dre-mir-430a-18,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430c-18,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430b-4,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430a-15,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430c-18,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430b-5,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430a-10,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430c-18,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430b-5,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430a-15,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430c-18,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430b-3,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430a-10,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430c-18,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430b-8,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430a-15,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430c-18,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430b-5,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430a-17,</span>
<span 
class="ptmr7t-x-x-85">miR-430,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430b-20,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430a-10,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430c-18,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430b-5,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430i-3,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430c-18,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430b-3,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430a-10,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430c-18,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430b-8,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430a-11,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430c-18,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430b-5,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430i-3,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430c-18,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430b-19,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430a-10,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430c-18,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430b-5,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430a-17,</span>
<span 
class="ptmr7t-x-x-85">miR-430,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430b-20,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430a-10,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430c-18,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430b-5,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430i-3,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430c-18,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430b-19,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430a-10,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430c-18,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430b-5,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430a-15,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430c-18,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430b-3,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430a-10,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430c-18,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430b-8,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430a-15,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430c-18,</span>
<span 
class="ptmr7t-x-x-85">dre-mir-430b-5</span>          </p></td>
                                                               

                                                               
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-14-"><td  style="white-space:wrap; text-align:left;" id="TBL-1-14-1"  
class="td11">
<!--l. 524--><p class="noindent" ><span 
class="ptmri7t-x-x-85">L. chalumnae</span>     </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-14-2"  
class="td11">
<!--l. 524--><p class="noindent" ><span 
class="ptmr7t-x-x-85">JH126646.1</span>       </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-14-3"  
class="td11">
<!--l. 524--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">1529355</span>          </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-14-4"  
class="td11">
<!--l. 524--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">1882777</span>          </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-14-5"  
class="td11">
<!--l. 524--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">353422</span>   </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-14-6"  
class="td11">
<!--l. 524--><p class="noindent" ><span 
class="zptmcm7t-x-x-85">7</span>         </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-1-14-7"  
class="td11">
<!--l. 525--><p class="noindent" ><span 
class="ptmr7t-x-x-85">mir-233,       mir-233,</span>
<span 
class="ptmr7t-x-x-85">mir-233,       mir-598,</span>
<span 
class="ptmr7t-x-x-85">mir-672,      MIR535,</span>
<span 
class="ptmr7t-x-x-85">mir-233</span>                      </p></td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-1-15-"><td  style="white-space:wrap; text-align:left;" id="TBL-1-15-1"  
class="td11">           </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-16-"><td  style="white-space:wrap; text-align:left;" id="TBL-1-16-1"  
class="td11">
<!--l. 526--><p class="noindent" >           </p></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-17-"><td  style="white-space:wrap; text-align:left;" id="TBL-1-17-1"  
class="td11">           </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-18-"><td  style="white-space:wrap; text-align:left;" id="TBL-1-18-1"  
class="td11">           </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-19-"><td  style="white-space:wrap; text-align:left;" id="TBL-1-19-1"  
class="td11">
<!--l. 527--><p class="noindent" >           </p></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-20-"><td  style="white-space:wrap; text-align:left;" id="TBL-1-20-1"  
class="td11">
<!--l. 527--><p class="noindent" >           </p></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-21-"><td  style="white-space:wrap; text-align:left;" id="TBL-1-21-1"  
class="td11">           </td><td  style="white-space:wrap; text-align:left;" id="TBL-1-21-2"  
class="td11">           </td><td  style="white-space:wrap; text-align:left;" id="TBL-1-21-3"  
class="td11">           </td><td  style="white-space:wrap; text-align:left;" id="TBL-1-21-4"  
class="td11">           </td><td  style="white-space:wrap; text-align:left;" id="TBL-1-21-5"  
class="td11">       </td><td  style="white-space:wrap; text-align:left;" id="TBL-1-21-6"  
class="td11">     </td><td  style="white-space:wrap; text-align:left;" id="TBL-1-21-7"  
class="td11"></td></tr>
</table></div>
</div>
<!--l. 531--><p class="noindent" >In <span 
class="ptmri7t-">C. robusta </span>some miRNAs are located in introns and a small class of miRNAs are derived
from the exonic parts of mRNAs (<a 
href="#XHendrix2010">Hendrix et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHendrix2010">2010</a>). In contrast, some miRNAs of in
<span 
class="ptmri7t-">O. dioca </span>are expressed from loci on the opposite strand on protein-coding genes. An
example is miR-2246, which also produces antisense miRs derived from miRNA loci
(moRs). Only <span 
class="zptmcm7t-">44 </span>loci appeared to be expressed as antisense products from the <span 
class="zptmcm7t-">300 </span>miRNA
loci predicted in 2010 by <a 
href="#XHendrix2010">Hendrix et&#x00A0;al.</a>&#x00A0;(<a 
href="#XHendrix2010">2010</a>). In Cionas have been also detected
miRNAs organized in clusters, for example in <span 
class="ptmri7t-">C. robusta </span>a putative cluster was detected
by <a 
href="#XKeshavan2010">Keshavan et&#x00A0;al.</a>&#x00A0;(<a 
href="#XKeshavan2010">2010</a>) using microarray analysis that shows a similar loci
organization to the cluster let-7/miR-125/miR-100 observed in Drosophila. The
miR-1473 was later classified as the orthologue of miR-100 in the analysis derived
from the comparison of the evolution of this cluster conducted by <a 
href="#XGriffiths-Jones2011">Griffiths-Jones
et&#x00A0;al.</a>&#x00A0;(<a 
href="#XGriffiths-Jones2011">2011</a>). The authors suggested that mir-100, mir125 and let7 are clustered
in most of the bilaterian genomes including the 1473 that is an orthologue of
mir-100.
</p><!--l. 539--><p class="indent" >  Current analysis of this cluster shows that the distribution of miRNAs families on this
let-7 cluster are present in all the studied chordate species. Vertebrate species such as
(<span 
class="ptmri7t-">D. rerio </span>and <span 
class="ptmri7t-">L. chalumnae</span>) contain more than one let-7 cluster, extending the
loci definition which is not restricted only to one element but for a cluster of
many loci with different length distributions. It is important to see that let-7 is
organized sometimes with another let-7 locus or with another miRNA&#8217;s loci families.
The distribution of this cluster reported in amphioxus is composed of 2 let-7
and 3 mir-10 (1 bfl-mir-100, 1 bfl-mir-125a and 1 bfl-mir-125b), this cluster
architecture is conserved in vertebrates that apparently inverted the order and split the
relation between let-7 and mir-10, creating two different cluster order groups:
let-7 + mir-10 and let-7 + other families. In this way, tunicates belong the latter
group, not including mir-10 on the cluster but including mir-233, mir-1473 or
mir-125.
</p><!--l. 554--><p class="indent" >  A second miRNA cluster consisting of the miR-182 and miR-183 was also detected in
                                                               

                                                               
<span 
class="ptmri7t-">C. robusta </span>in 2010 by <a 
href="#XKeshavan2010">Keshavan et&#x00A0;al.</a>&#x00A0;(<a 
href="#XKeshavan2010">2010</a>) which in the current predictions reported
another member locus: the miR-96 organized in the middle of those loci, as is shown in
the plot <a 
href="#x1-8015r7">7<!--tex4ht:ref: fig:mir-96 --></a>. Here the authors also found five additional paralogs of let-7 within a
1-kb stretch, but it is important to know that those elements had been identified
on chromosome 4q on Ensembl release 54 version, in the current version only
two of those elements have been identified by homology approaches (Figure
<a 
href="#x1-8003r1">1<!--tex4ht:ref: fig:let-7 --></a>).
</p><!--l. 564--><p class="indent" >  The cluster miR-1/miR-133, expressed specifically in Cionas muscle tissues was also
reported by <a 
href="#XKusakabe2013">Kusakabe et&#x00A0;al.</a>&#x00A0;(<a 
href="#XKusakabe2013">2013</a>). The authors reported that one copy of this cluster is
presented in both Cionas. As is shown in the plot <a 
href="#x1-8005r2">2<!--tex4ht:ref: fig:mir-1 --></a> a copy is also present in <span 
class="ptmri7t-">L. chalumnae</span>.
In 2012 a new cluster was proposed in <span 
class="ptmri7t-">C. robusta </span>by <a 
href="#XTerai2012">Terai et&#x00A0;al.</a>&#x00A0;(<a 
href="#XTerai2012">2012</a>) located on the
chromosome 10q and composed of the mir-4054 locus and the mir-4091. In the current
distribution of this cluster a new annotated family the mir-4008 with three paralogous is
located on the middle of those loci. This current distribution is shown in Table <a 
href="#x1-8017r2">2<!--tex4ht:ref: tab:otherClusters --></a>
these loci were validated by <a 
href="#XNorden-Krichmar2007">Norden-Krichmar et&#x00A0;al.</a>&#x00A0;(<a 
href="#XNorden-Krichmar2007">2007</a>), <a 
href="#XFu:08">Fu et&#x00A0;al.</a>&#x00A0;(<a 
href="#XFu:08">2008</a>),
<a 
href="#XHendrix2010">Hendrix et&#x00A0;al.</a>&#x00A0;(<a 
href="#XHendrix2010">2010</a>), and <a 
href="#XTerai2012">Terai et&#x00A0;al.</a>&#x00A0;(<a 
href="#XTerai2012">2012</a>). As was mentioned by <a 
href="#XHendrix2010">Hendrix
et&#x00A0;al.</a>&#x00A0;(<a 
href="#XHendrix2010">2010</a>) it is not very common to find related miRNAs organized in clusters
composed by closely related families that differ in just a single nucleotide in the seed
sequence as was found on the cluster composed by nine Ci-mir-2200, seven
Ci-mir-2201 and nine Ci-mir2203 which were previously reported under that
putative names. The same authors also found a second large cluster composed of
<span 
class="zptmcm7t-">11 </span>miRNAs that gather into <span 
class="zptmcm7t-">4 </span>paralogous families three of Ci-mir-2200, three
Ci-mir-2201, four Ci-mir-2204 and two Ci-2217. Current distribution of miRNAs
families in <span 
class="ptmri7t-">C. robusta </span>and curated annotations indicate than in other regions of the
chromosome 7q the miRNAs are organized in tandem copies of families. An example is
the big cluster composed of the families miR4000, miR-4001, miR4002, and
miR4006 located on chromosome 7q (Table <a 
href="#x1-8017r2">2<!--tex4ht:ref: tab:otherClusters --></a>). Another cluster is also located on the
same chromosome composed by the families miR-4003, miR4005 and miR4077
in (Table <a 
href="#x1-8017r2">2<!--tex4ht:ref: tab:otherClusters --></a>). Some other clusters are also found on the chromosome 1a, 10q
and 3p. See this structure in Table <a 
href="#x1-8017r2">2<!--tex4ht:ref: tab:otherClusters --></a>, most of them were validated by <a 
href="#XHendrix2010">Hendrix
et&#x00A0;al.</a>&#x00A0;(<a 
href="#XHendrix2010">2010</a>).
</p><!--l. 591--><p class="indent" >  Some other clusters shared between both Cionas are the cluster 92, 124 and 200
validated by <a 
href="#XNorden-Krichmar2007">Norden-Krichmar et&#x00A0;al.</a>&#x00A0;(<a 
href="#XNorden-Krichmar2007">2007</a>), <a 
href="#XFu:08">Fu et&#x00A0;al.</a>&#x00A0;(<a 
href="#XFu:08">2008</a>), and <a 
href="#XHendrix2010">Hendrix
et&#x00A0;al.</a>&#x00A0;(<a 
href="#XHendrix2010">2010</a>), which structure is shown in Table <a 
href="#x1-8017r2">2<!--tex4ht:ref: tab:otherClusters --></a>.
</p>
  <hr class="figure" /><div class="figure" 
>
                                                               

                                                               
                                                               

                                                               
<a 
 id="x1-8003r1"></a>
<a 
 id="x1-8004"></a>
<span 
class="ptmb7t-x-x-85">Fig.</span><span 
class="ptmb7t-x-x-85">&#x00A0;1  </span><span 
class="ptmr7t-x-x-85">Multiple alignment</span>
<span 
class="ptmr7t-x-x-85">of  let-7  clusters.  Specific</span>
<span 
class="ptmr7t-x-x-85">names  from  annotations</span>
<span 
class="ptmr7t-x-x-85">and homology predictions</span>
<span 
class="ptmr7t-x-x-85">are described in the legend.</span>
<span 
class="ptmr7t-x-x-85">Names  from  miRBase</span>
<span 
class="ptmr7t-x-x-85">families  are  reported  at</span>
<span 
class="ptmr7t-x-x-85">the bottom of the aligned</span>
<span 
class="ptmr7t-x-x-85">elements.</span>
                                                               

                                                               
  </div><hr class="endfigure" />
  <hr class="figure" /><div class="figure" 
>
                                                               

                                                               
                                                               

                                                               
<a 
 id="x1-8005r2"></a>
<a 
 id="x1-8006"></a>
<span 
class="ptmb7t-x-x-85">Fig.</span><span 
class="ptmb7t-x-x-85">&#x00A0;2  </span><span 
class="ptmr7t-x-x-85">mir-1/mir-133</span>
                                                               

                                                               
  </div><hr class="endfigure" />
  <hr class="figure" /><div class="figure" 
>
                                                               

                                                               
                                                               

                                                               
<a 
 id="x1-8007r3"></a>
<a 
 id="x1-8008"></a>
<span 
class="ptmb7t-x-x-85">Fig.</span><span 
class="ptmb7t-x-x-85">&#x00A0;3  </span><span 
class="ptmr7t-x-x-85">mir-181</span>
                                                               

                                                               
  </div><hr class="endfigure" />
  <hr class="figure" /><div class="figure" 
>
                                                               

                                                               
                                                               

                                                               
<a 
 id="x1-8009r4"></a>
<a 
 id="x1-8010"></a>
<span 
class="ptmb7t-x-x-85">Fig.</span><span 
class="ptmb7t-x-x-85">&#x00A0;4  </span><span 
class="ptmr7t-x-x-85">mir-216/mir-217</span>
                                                               

                                                               
  </div><hr class="endfigure" />
  <hr class="figure" /><div class="figure" 
>
                                                               

                                                               
                                                               

                                                               
<a 
 id="x1-8011r5"></a>
<a 
 id="x1-8012"></a>
<span 
class="ptmb7t-x-x-85">Fig.</span><span 
class="ptmb7t-x-x-85">&#x00A0;5  </span><span 
class="ptmr7t-x-x-85">mir-34</span>
                                                               

                                                               
  </div><hr class="endfigure" />
  <hr class="figure" /><div class="figure" 
>
                                                               

                                                               
                                                               

                                                               
<a 
 id="x1-8013r6"></a>
<a 
 id="x1-8014"></a>
<span 
class="ptmb7t-x-x-85">Fig.</span><span 
class="ptmb7t-x-x-85">&#x00A0;6  </span><span 
class="ptmr7t-x-x-85">mir-92</span>
                                                               

                                                               
  </div><hr class="endfigure" />
  <hr class="figure" /><div class="figure" 
>
                                                               

                                                               
                                                               

                                                               
<a 
 id="x1-8015r7"></a>
<a 
 id="x1-8016"></a>
<span 
class="ptmb7t-x-x-85">Fig.</span><span 
class="ptmb7t-x-x-85">&#x00A0;7  </span><span 
class="ptmr7t-x-x-85">mir-182/mir-96/mir-183</span>
                                                               

                                                               
  </div><hr class="endfigure" />
<div class="center" 
>
<!--l. 657--><p class="noindent" >
<a 
 id="x1-8017r2"></a>
</p><!--l. 659--><div class="longtable"> <table id="TBL-2" class="longtable" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-2-1g"><col 
id="TBL-2-1" /><col 
id="TBL-2-2" /><col 
id="TBL-2-3" /><col 
id="TBL-2-4" /><col 
id="TBL-2-5" /><col 
id="TBL-2-6" /><col 
id="TBL-2-7" /><col 
id="TBL-2-8" /></colgroup>
<tr  
 style="vertical-align:baseline;" id="TBL-2-1-"><td  style="white-space:nowrap; text-align:center;" id="TBL-2-1-1"  
class="td11">     </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-2-"><td colspan="3" style="white-space:nowrap; text-align:center;" id="TBL-2-2-1"  
class="td11"></td>             <div class="multicolumn"  style="white-space:nowrap; text-align:center;"> <div class="caption" 
><span class="id"><span 
class="ptmr7t-x-x-85">Table</span><span 
class="ptmr7t-x-x-85">&#x00A0;2:  </span></span><span  
class="content"><span 
class="ptmr7t-x-x-85">Reported  clusters  in  the  literature.  Bold  text  represent  those  miRNAs</span>
<span 
class="ptmr7t-x-x-85">elements  that  are  currently  annotated  and  validated,  but  could  not  be  detected  by</span>
<span 
class="ptmr7t-x-x-85">homology strategies. </span><span 
class="ptmb7t-x-x-85">Ciro</span><span 
class="ptmr7t-x-x-85">: </span><span 
class="ptmri7t-x-x-85">C. robusta</span><span 
class="ptmr7t-x-x-85">, </span><span 
class="ptmb7t-x-x-85">Cisa</span><span 
class="ptmr7t-x-x-85">: </span><span 
class="ptmri7t-x-x-85">C. savignyi </span><span 
class="ptmr7t-x-x-85">and </span><span 
class="ptmb7t-x-x-85">Oidi</span><span 
class="ptmr7t-x-x-85">: </span><span 
class="ptmri7t-x-x-85">O. dioica</span></span></div><!--tex4ht:label?: x1-8017r2 -->             </div>                               <a 
 id="x1-8018"></a>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-3-"><td  style="white-space:nowrap; text-align:center;" id="TBL-2-3-1"  
class="td11">
</td></tr><tr 
class="cline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-2-4-"><td  style="white-space:nowrap; text-align:center;" id="TBL-2-4-1"  
class="td11"><span 
class="ptmb7t-x-x-85">Specie</span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-4-2"  
class="td11"><span 
class="ptmb7t-x-x-85">Chr             </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-4-3"  
class="td11"><span 
class="ptmb7t-x-x-85">Start     </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-4-4"  
class="td11"><span 
class="ptmb7t-x-x-85">End       </span></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-4-5"  
class="td11">
<!--l. 672--><p class="noindent" ><span 
class="ptmb7t-x-x-85">miRNAs</span>                   </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-4-6"  
class="td11">
<!--l. 672--><p class="noindent" ><span 
class="ptmb7t-x-x-85">Comments</span>                      </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-4-7"  
class="td11">
<!--l. 672--><p class="noindent" ><span 
class="ptmb7t-x-x-85">Source DB</span>  </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-4-8"  
class="td11">
<!--l. 673--><p class="noindent" ><span 
class="ptmb7t-x-x-85">Ref.</span>      </p></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-5-"><td  style="white-space:nowrap; text-align:center;" id="TBL-2-5-1"  
class="td11">     </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-6-"><td  style="white-space:nowrap; text-align:center;" id="TBL-2-6-1"  
class="td11">
</td></tr><tr 
class="cline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-2-7-"><td  style="white-space:nowrap; text-align:center;" id="TBL-2-7-1"  
class="td11"> <span 
class="ptmr7t-x-x-85">Ciro  </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-7-2"  
class="td11"><span 
class="ptmr7t-x-x-85">4q                </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-7-3"  
class="td11"><span 
class="zptmcm7t-x-x-85">2082260</span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-7-4"  
class="td11"><span 
class="zptmcm7t-x-x-85">2083286</span></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-7-5"  
class="td11">
<!--l. 677--><p class="noindent" ><span 
class="ptmb7t-x-x-85">cin-let-7a-1,</span>
<span 
class="ptmb7t-x-x-85">cin-let-7f,</span>
<span 
class="ptmb7t-x-x-85">cin-let-7b,</span>
<span 
class="ptmb7t-x-x-85">cin-let-7c,</span>
<span 
class="ptmb7t-x-x-85">cin-let-7a-2</span>              </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-7-6"  
class="td11">
<!--l. 678--><p class="noindent" ><span 
class="ptmr7t-x-x-85">Reported                      on</span>
<span 
class="ptmr7t-x-x-85">miRBAse  and  annotated</span>
<span 
class="ptmr7t-x-x-85">on Ensembl</span>                     </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-7-7"  
class="td11">
<!--l. 678--><p class="noindent" ><span 
class="ptmr7t-x-x-85">miRBase</span>     </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-7-8"  
class="td11">
<!--l. 678--><p class="noindent" ><span 
class="ptmr7t-x-x-85">(</span><a 
href="#XHendrix2010"><span 
class="ptmr7t-x-x-85">Hendrix</span>
<span 
class="ptmr7t-x-x-85">et</span><span 
class="ptmr7t-x-x-85">&#x00A0;al.</span></a><span 
class="ptmr7t-x-x-85">,</span><span 
class="ptmr7t-x-x-85">&#x00A0;</span><a 
href="#XHendrix2010"><span 
class="ptmr7t-x-x-85">2010</span></a><span 
class="ptmr7t-x-x-85">),</span>
<span 
class="ptmr7t-x-x-85">(</span><a 
href="#XFu:08"><span 
class="ptmr7t-x-x-85">Fu</span>
<span 
class="ptmr7t-x-x-85">et</span><span 
class="ptmr7t-x-x-85">&#x00A0;al.</span></a><span 
class="ptmr7t-x-x-85">,</span><span 
class="ptmr7t-x-x-85">&#x00A0;</span><a 
href="#XFu:08"><span 
class="ptmr7t-x-x-85">2008</span></a><span 
class="ptmr7t-x-x-85">)</span> </p> </td>
                                                               

                                                               
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-8-"><td  style="white-space:nowrap; text-align:center;" id="TBL-2-8-1"  
class="td11"> <span 
class="ptmr7t-x-x-85">Cisa  </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-8-2"  
class="td11"><span 
class="ptmr7t-x-x-85">reftig</span><span 
class="ptmr7t-x-x-85">_41      </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-8-3"  
class="td11"><span 
class="zptmcm7t-x-x-85">1114139</span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-8-4"  
class="td11"><span 
class="zptmcm7t-x-x-85">1117597</span></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-8-5"  
class="td11">
<!--l. 681--><p class="noindent" ><span 
class="ptmb7t-x-x-85">csa-let-7c-1,</span>
<span 
class="ptmb7t-x-x-85">csa-let-7b,</span>
<span 
class="ptmb7t-x-x-85">csa-let-7a,</span>
<span 
class="ptmb7t-x-x-85">csa-let-7c-2</span>              </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-8-6"  
class="td11">
<!--l. 681--><p class="noindent" ><span 
class="ptmr7t-x-x-85">Reported   on   miRBase</span>
<span 
class="ptmr7t-x-x-85">and does not detected by</span>
<span 
class="ptmr7t-x-x-85">homology strategies.</span>       </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-8-7"  
class="td11">
<!--l. 682--><p class="noindent" ><span 
class="ptmr7t-x-x-85">miRBase</span>     </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-8-8"  
class="td11">
<!--l. 682--><p class="noindent" ><span 
class="ptmr7t-x-x-85">(</span><a 
href="#XFu:08"><span 
class="ptmr7t-x-x-85">Fu</span>
<span 
class="ptmr7t-x-x-85">et</span><span 
class="ptmr7t-x-x-85">&#x00A0;al.</span></a><span 
class="ptmr7t-x-x-85">,</span><span 
class="ptmr7t-x-x-85">&#x00A0;</span><a 
href="#XFu:08"><span 
class="ptmr7t-x-x-85">2008</span></a><span 
class="ptmr7t-x-x-85">)</span> </p> </td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-2-9-"><td  style="white-space:nowrap; text-align:center;" id="TBL-2-9-1"  
class="td11"> <span 
class="ptmr7t-x-x-85">Ciro  </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-9-2"  
class="td11"><span 
class="ptmr7t-x-x-85">10q              </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-9-3"  
class="td11"><span 
class="zptmcm7t-x-x-85">3226200</span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-9-4"  
class="td11"><span 
class="zptmcm7t-x-x-85">3228884</span></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-9-5"  
class="td11">
<!--l. 684--><p class="noindent" ><span 
class="ptmb7t-x-x-85">cin-mir-34</span><span 
class="ptmr7t-x-x-85">,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4091,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4008a,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4008c,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4008b,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4054</span>            </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-9-6"  
class="td11">
<!--l. 685--><p class="noindent" ><span 
class="ptmr7t-x-x-85">NA</span>                                   </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-9-7"  
class="td11">
<!--l. 686--><p class="noindent" ><span 
class="ptmr7t-x-x-85">miRBase,</span>
<span 
class="ptmr7t-x-x-85">Homology</span>   </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-9-8"  
class="td11">
<!--l. 686--><p class="noindent" ><span 
class="ptmr7t-x-x-85">(</span><a 
href="#XNorden-Krichmar2007"><span 
class="ptmr7t-x-x-85">Norden-Krichmar</span>
<span 
class="ptmr7t-x-x-85">et</span><span 
class="ptmr7t-x-x-85">&#x00A0;al.</span></a><span 
class="ptmr7t-x-x-85">,</span><span 
class="ptmr7t-x-x-85">&#x00A0;</span><a 
href="#XNorden-Krichmar2007"><span 
class="ptmr7t-x-x-85">2007</span></a><span 
class="ptmr7t-x-x-85">),</span>
<span 
class="ptmr7t-x-x-85">(</span><a 
href="#XFu:08"><span 
class="ptmr7t-x-x-85">Fu</span>
<span 
class="ptmr7t-x-x-85">et</span><span 
class="ptmr7t-x-x-85">&#x00A0;al.</span></a><span 
class="ptmr7t-x-x-85">,</span><span 
class="ptmr7t-x-x-85">&#x00A0;</span><a 
href="#XFu:08"><span 
class="ptmr7t-x-x-85">2008</span></a><span 
class="ptmr7t-x-x-85">),</span>
<span 
class="ptmr7t-x-x-85">(</span><a 
href="#XHendrix2010"><span 
class="ptmr7t-x-x-85">Hendrix</span>
<span 
class="ptmr7t-x-x-85">et</span><span 
class="ptmr7t-x-x-85">&#x00A0;al.</span></a><span 
class="ptmr7t-x-x-85">,</span><span 
class="ptmr7t-x-x-85">&#x00A0;</span><a 
href="#XHendrix2010"><span 
class="ptmr7t-x-x-85">2010</span></a><span 
class="ptmr7t-x-x-85">),</span>
<span 
class="ptmr7t-x-x-85">(</span><a 
href="#XTerai2012"><span 
class="ptmr7t-x-x-85">Terai</span>
<span 
class="ptmr7t-x-x-85">et</span><span 
class="ptmr7t-x-x-85">&#x00A0;al.</span></a><span 
class="ptmr7t-x-x-85">,</span><span 
class="ptmr7t-x-x-85">&#x00A0;</span><a 
href="#XTerai2012"><span 
class="ptmr7t-x-x-85">2012</span></a><span 
class="ptmr7t-x-x-85">)</span> </p> </td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-2-10-"><td  style="white-space:nowrap; text-align:center;" id="TBL-2-10-1"  
class="td11"> <span 
class="ptmr7t-x-x-85">Ciro  </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-10-2"  
class="td11"><span 
class="ptmr7t-x-x-85">7q                </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-10-3"  
class="td11"><span 
class="zptmcm7t-x-x-85">4828431</span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-10-4"  
class="td11"><span 
class="zptmcm7t-x-x-85">4835967</span></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-10-5"  
class="td11">
<!--l. 689--><p class="noindent" ><span 
class="ptmr7t-x-x-85">cin-mir-4077b,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4003b,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4005b,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4077d,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4003a-1,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4003c,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4077a,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4003a-4,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4003d</span>          </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-10-6"  
class="td11">
<!--l. 691--><p class="noindent" ><span 
class="ptmr7t-x-x-85">NA</span>                                   </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-10-7"  
class="td11">
<!--l. 691--><p class="noindent" ><span 
class="ptmr7t-x-x-85">miRBase,</span>
<span 
class="ptmr7t-x-x-85">Homology</span>   </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-10-8"  
class="td11">
<!--l. 692--><p class="noindent" ><span 
class="ptmr7t-x-x-85">(</span><a 
href="#XHendrix2010"><span 
class="ptmr7t-x-x-85">Hendrix</span>
<span 
class="ptmr7t-x-x-85">et</span><span 
class="ptmr7t-x-x-85">&#x00A0;al.</span></a><span 
class="ptmr7t-x-x-85">,</span><span 
class="ptmr7t-x-x-85">&#x00A0;</span><a 
href="#XHendrix2010"><span 
class="ptmr7t-x-x-85">2010</span></a><span 
class="ptmr7t-x-x-85">)</span> </p> </td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-2-11-"><td  style="white-space:nowrap; text-align:center;" id="TBL-2-11-1"  
class="td11"> <span 
class="ptmr7t-x-x-85">Ciro  </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-11-2"  
class="td11"><span 
class="ptmr7t-x-x-85">3q                </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-11-3"  
class="td11"><span 
class="zptmcm7t-x-x-85">567478  </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-11-4"  
class="td11"><span 
class="zptmcm7t-x-x-85">571031  </span></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-11-5"  
class="td11">
<!--l. 694--><p class="noindent" ><span 
class="ptmr7t-x-x-85">cin-mir-4001f,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4000e,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4001c,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-1502d,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4018a,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4019,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-1502b,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-1502a,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4007,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4000f,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4001i,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4018b,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-1502c</span>           </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-11-6"  
class="td11">
<!--l. 697--><p class="noindent" ><span 
class="ptmr7t-x-x-85">Inclusion                      of</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4019             and</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4007</span>                   </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-11-7"  
class="td11">
<!--l. 697--><p class="noindent" ><span 
class="ptmr7t-x-x-85">miRBase,</span>
<span 
class="ptmr7t-x-x-85">Homology</span>   </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-11-8"  
class="td11">
<!--l. 698--><p class="noindent" ><span 
class="ptmr7t-x-x-85">(</span><a 
href="#XHendrix2010"><span 
class="ptmr7t-x-x-85">Hendrix</span>
<span 
class="ptmr7t-x-x-85">et</span><span 
class="ptmr7t-x-x-85">&#x00A0;al.</span></a><span 
class="ptmr7t-x-x-85">,</span><span 
class="ptmr7t-x-x-85">&#x00A0;</span><a 
href="#XHendrix2010"><span 
class="ptmr7t-x-x-85">2010</span></a><span 
class="ptmr7t-x-x-85">)</span> </p> </td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-2-12-"><td  style="white-space:nowrap; text-align:center;" id="TBL-2-12-1"  
class="td11"> <span 
class="ptmr7t-x-x-85">Ciro  </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-12-2"  
class="td11"><span 
class="ptmr7t-x-x-85">HT000037.1</span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-12-3"  
class="td11"><span 
class="zptmcm7t-x-x-85">4884     </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-12-4"  
class="td11"><span 
class="zptmcm7t-x-x-85">5250     </span></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-12-5"  
class="td11">
<!--l. 700--><p class="noindent" ><span 
class="ptmb7t-x-x-85">cin-mir-367</span><span 
class="ptmr7t-x-x-85">,</span>
<span 
class="ptmb7t-x-x-85">cin-mir-4009c</span><span 
class="ptmr7t-x-x-85">,</span>
<span 
class="ptmb7t-x-x-85">cin-mir-4009b</span><span 
class="ptmr7t-x-x-85">,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-367,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4009c,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4009a,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4009b</span>          </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-12-6"  
class="td11">
<!--l. 702--><p class="noindent" ><span 
class="ptmr7t-x-x-85">Non-highlighted</span>
<span 
class="ptmr7t-x-x-85">names   could   be   found</span>
<span 
class="ptmr7t-x-x-85">in the current genome at</span>
<span 
class="ptmr7t-x-x-85">HT000037.1 scaffold</span>      </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-12-7"  
class="td11">
<!--l. 703--><p class="noindent" ><span 
class="ptmr7t-x-x-85">miRBase,</span>
<span 
class="ptmr7t-x-x-85">Homology</span>   </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-12-8"  
class="td11">
<!--l. 704--><p class="noindent" ><span 
class="ptmr7t-x-x-85">(</span><a 
href="#XHendrix2010"><span 
class="ptmr7t-x-x-85">Hendrix</span>
<span 
class="ptmr7t-x-x-85">et</span><span 
class="ptmr7t-x-x-85">&#x00A0;al.</span></a><span 
class="ptmr7t-x-x-85">,</span><span 
class="ptmr7t-x-x-85">&#x00A0;</span><a 
href="#XHendrix2010"><span 
class="ptmr7t-x-x-85">2010</span></a><span 
class="ptmr7t-x-x-85">)</span> </p> </td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-2-13-"><td  style="white-space:nowrap; text-align:center;" id="TBL-2-13-1"  
class="td11"> <span 
class="ptmr7t-x-x-85">Ciro  </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-13-2"  
class="td11"><span 
class="ptmr7t-x-x-85">10q              </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-13-3"  
class="td11"><span 
class="zptmcm7t-x-x-85">3226200</span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-13-4"  
class="td11"><span 
class="zptmcm7t-x-x-85">3228884</span></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-13-5"  
class="td11">
<!--l. 706--><p class="noindent" ><span 
class="ptmb7t-x-x-85">cin-mir-34</span><span 
class="ptmr7t-x-x-85">,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4091,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4008a,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4008c,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4008b,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4054</span>            </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-13-6"  
class="td11">
<!--l. 707--><p class="noindent" ><span 
class="ptmr7t-x-x-85">NA</span>                                   </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-13-7"  
class="td11">
<!--l. 707--><p class="noindent" ><span 
class="ptmr7t-x-x-85">miRBase,</span>
<span 
class="ptmr7t-x-x-85">Homology</span>   </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-13-8"  
class="td11">
<!--l. 708--><p class="noindent" ><span 
class="ptmr7t-x-x-85">(</span><a 
href="#XHendrix2010"><span 
class="ptmr7t-x-x-85">Hendrix</span>
<span 
class="ptmr7t-x-x-85">et</span><span 
class="ptmr7t-x-x-85">&#x00A0;al.</span></a><span 
class="ptmr7t-x-x-85">,</span><span 
class="ptmr7t-x-x-85">&#x00A0;</span><a 
href="#XHendrix2010"><span 
class="ptmr7t-x-x-85">2010</span></a><span 
class="ptmr7t-x-x-85">)</span> </p> </td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-2-14-"><td  style="white-space:nowrap; text-align:center;" id="TBL-2-14-1"  
class="td11"> <span 
class="ptmr7t-x-x-85">Ciro  </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-14-2"  
class="td11"><span 
class="ptmr7t-x-x-85">3q                </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-14-3"  
class="td11"><span 
class="zptmcm7t-x-x-85">884615  </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-14-4"  
class="td11"><span 
class="zptmcm7t-x-x-85">885508  </span></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-14-5"  
class="td11">
<!--l. 711--><p class="noindent" ><span 
class="ptmr7t-x-x-85">cin-mir-92a,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-92d,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-92c</span>               </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-14-6"  
class="td11">
<!--l. 711--><p class="noindent" ><span 
class="ptmr7t-x-x-85">NA</span>                                   </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-14-7"  
class="td11">
<!--l. 711--><p class="noindent" ><span 
class="ptmr7t-x-x-85">miRBase,</span>
<span 
class="ptmr7t-x-x-85">Homology</span>   </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-14-8"  
class="td11">
<!--l. 712--><p class="noindent" ><span 
class="ptmr7t-x-x-85">(</span><a 
href="#XHendrix2010"><span 
class="ptmr7t-x-x-85">Hendrix</span>
<span 
class="ptmr7t-x-x-85">et</span><span 
class="ptmr7t-x-x-85">&#x00A0;al.</span></a><span 
class="ptmr7t-x-x-85">,</span><span 
class="ptmr7t-x-x-85">&#x00A0;</span><a 
href="#XHendrix2010"><span 
class="ptmr7t-x-x-85">2010</span></a><span 
class="ptmr7t-x-x-85">)</span> </p> </td>
                                                               

                                                               
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-15-"><td  style="white-space:nowrap; text-align:center;" id="TBL-2-15-1"  
class="td11"> <span 
class="ptmr7t-x-x-85">Cisa  </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-15-2"  
class="td11"><span 
class="ptmr7t-x-x-85">reftig</span><span 
class="ptmr7t-x-x-85">_1        </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-15-3"  
class="td11"><span 
class="zptmcm7t-x-x-85">1335375</span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-15-4"  
class="td11"><span 
class="zptmcm7t-x-x-85">1336487</span></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-15-5"  
class="td11">
<!--l. 713--><p class="noindent" ><span 
class="ptmr7t-x-x-85">csa-mir-92b,</span>
<span 
class="ptmr7t-x-x-85">csa-mir-92c,</span>
<span 
class="ptmr7t-x-x-85">csa-mir-92a</span>              </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-15-6"  
class="td11">
<!--l. 714--><p class="noindent" ><span 
class="ptmr7t-x-x-85">NA</span>                                   </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-15-7"  
class="td11">
<!--l. 714--><p class="noindent" ><span 
class="ptmr7t-x-x-85">miRBase,</span>
<span 
class="ptmr7t-x-x-85">Homology</span>   </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-15-8"  
class="td11">
<!--l. 714--><p class="noindent" ><span 
class="ptmr7t-x-x-85">(</span><a 
href="#XFu:08"><span 
class="ptmr7t-x-x-85">Fu</span>
<span 
class="ptmr7t-x-x-85">et</span><span 
class="ptmr7t-x-x-85">&#x00A0;al.</span></a><span 
class="ptmr7t-x-x-85">,</span><span 
class="ptmr7t-x-x-85">&#x00A0;</span><a 
href="#XFu:08"><span 
class="ptmr7t-x-x-85">2008</span></a><span 
class="ptmr7t-x-x-85">)</span> </p> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-16-"><td  style="white-space:nowrap; text-align:center;" id="TBL-2-16-1"  
class="td11"> <span 
class="ptmr7t-x-x-85">Oidi  </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-16-2"  
class="td11"><span 
class="ptmr7t-x-x-85">scaffold</span><span 
class="ptmr7t-x-x-85">_1    </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-16-3"  
class="td11"><span 
class="zptmcm7t-x-x-85">3086369</span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-16-4"  
class="td11"><span 
class="zptmcm7t-x-x-85">3086586</span></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-16-5"  
class="td11">
<!--l. 715--><p class="noindent" ><span 
class="ptmr7t-x-x-85">odi-mir-92b,</span>
<span 
class="ptmr7t-x-x-85">odi-mir-92a</span>              </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-16-6"  
class="td11">
<!--l. 715--><p class="noindent" ><span 
class="ptmr7t-x-x-85">NA</span>                                   </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-16-7"  
class="td11">
<!--l. 716--><p class="noindent" ><span 
class="ptmr7t-x-x-85">Homology</span>   </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-16-8"  
class="td11">
<!--l. 716--><p class="noindent" ><span 
class="ptmr7t-x-x-85">(</span><a 
href="#XFu:08"><span 
class="ptmr7t-x-x-85">Fu</span>
<span 
class="ptmr7t-x-x-85">et</span><span 
class="ptmr7t-x-x-85">&#x00A0;al.</span></a><span 
class="ptmr7t-x-x-85">,</span><span 
class="ptmr7t-x-x-85">&#x00A0;</span><a 
href="#XFu:08"><span 
class="ptmr7t-x-x-85">2008</span></a><span 
class="ptmr7t-x-x-85">)</span> </p> </td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-2-17-"><td  style="white-space:nowrap; text-align:center;" id="TBL-2-17-1"  
class="td11"> <span 
class="ptmr7t-x-x-85">Ciro  </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-17-2"  
class="td11"><span 
class="ptmr7t-x-x-85">7q                </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-17-3"  
class="td11"><span 
class="zptmcm7t-x-x-85">4969691</span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-17-4"  
class="td11"><span 
class="zptmcm7t-x-x-85">4969912</span></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-17-5"  
class="td11">
<!--l. 718--><p class="noindent" ><span 
class="ptmr7t-x-x-85">cin-mir-124-1,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-124-2</span>           </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-17-6"  
class="td11">
<!--l. 719--><p class="noindent" ><span 
class="ptmr7t-x-x-85">NA</span>                                   </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-17-7"  
class="td11">
<!--l. 719--><p class="noindent" ><span 
class="ptmr7t-x-x-85">miRBase,</span>
<span 
class="ptmr7t-x-x-85">Homology</span>   </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-17-8"  
class="td11">
<!--l. 719--><p class="noindent" ><span 
class="ptmr7t-x-x-85">(</span><a 
href="#XHendrix2010"><span 
class="ptmr7t-x-x-85">Hendrix</span>
<span 
class="ptmr7t-x-x-85">et</span><span 
class="ptmr7t-x-x-85">&#x00A0;al.</span></a><span 
class="ptmr7t-x-x-85">,</span><span 
class="ptmr7t-x-x-85">&#x00A0;</span><a 
href="#XHendrix2010"><span 
class="ptmr7t-x-x-85">2010</span></a><span 
class="ptmr7t-x-x-85">),</span>
<span 
class="ptmr7t-x-x-85">(</span><a 
href="#XFu:08"><span 
class="ptmr7t-x-x-85">Fu</span>
<span 
class="ptmr7t-x-x-85">et</span><span 
class="ptmr7t-x-x-85">&#x00A0;al.</span></a><span 
class="ptmr7t-x-x-85">,</span><span 
class="ptmr7t-x-x-85">&#x00A0;</span><a 
href="#XFu:08"><span 
class="ptmr7t-x-x-85">2008</span></a><span 
class="ptmr7t-x-x-85">)</span> </p> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-18-"><td  style="white-space:nowrap; text-align:center;" id="TBL-2-18-1"  
class="td11"> <span 
class="ptmr7t-x-x-85">Cisa  </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-18-2"  
class="td11"><span 
class="ptmr7t-x-x-85">reftig</span><span 
class="ptmr7t-x-x-85">_262    </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-18-3"  
class="td11"><span 
class="zptmcm7t-x-x-85">49392   </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-18-4"  
class="td11"><span 
class="zptmcm7t-x-x-85">49620   </span></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-18-5"  
class="td11">
<!--l. 720--><p class="noindent" ><span 
class="ptmr7t-x-x-85">csa-mir-124-1,</span>
<span 
class="ptmr7t-x-x-85">csa-mir-124-2</span>           </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-18-6"  
class="td11">
<!--l. 721--><p class="noindent" ><span 
class="ptmr7t-x-x-85">NA</span>                                   </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-18-7"  
class="td11">
<!--l. 721--><p class="noindent" ><span 
class="ptmr7t-x-x-85">miRBase,</span>
<span 
class="ptmr7t-x-x-85">Homology</span>   </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-18-8"  
class="td11">
<!--l. 721--><p class="noindent" ><span 
class="ptmr7t-x-x-85">(</span><a 
href="#XFu:08"><span 
class="ptmr7t-x-x-85">Fu</span>
<span 
class="ptmr7t-x-x-85">et</span><span 
class="ptmr7t-x-x-85">&#x00A0;al.</span></a><span 
class="ptmr7t-x-x-85">,</span><span 
class="ptmr7t-x-x-85">&#x00A0;</span><a 
href="#XFu:08"><span 
class="ptmr7t-x-x-85">2008</span></a><span 
class="ptmr7t-x-x-85">)</span> </p> </td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-2-19-"><td  style="white-space:nowrap; text-align:center;" id="TBL-2-19-1"  
class="td11"> <span 
class="ptmr7t-x-x-85">Ciro  </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-19-2"  
class="td11"><span 
class="ptmr7t-x-x-85">HT000325.1</span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-19-3"  
class="td11"><span 
class="zptmcm7t-x-x-85">8331     </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-19-4"  
class="td11"><span 
class="zptmcm7t-x-x-85">8778     </span></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-19-5"  
class="td11">
<!--l. 724--><p class="noindent" ><span 
class="ptmr7t-x-x-85">cin-mir-200,</span>
<span 
class="ptmb7t-x-x-85">cin-mir-3575</span><span 
class="ptmr7t-x-x-85">,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-141,</span>
<span 
class="ptmb7t-x-x-85">cin-mir-5611</span>            </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-19-6"  
class="td11">
<!--l. 724--><p class="noindent" ><span 
class="ptmr7t-x-x-85">NA</span>                                   </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-19-7"  
class="td11">
<!--l. 725--><p class="noindent" ><span 
class="ptmr7t-x-x-85">miRBase,</span>
<span 
class="ptmr7t-x-x-85">Homology</span>   </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-19-8"  
class="td11">
<!--l. 725--><p class="noindent" ><span 
class="ptmr7t-x-x-85">(</span><a 
href="#XNorden-Krichmar2007"><span 
class="ptmr7t-x-x-85">Norden-Krichmar</span>
<span 
class="ptmr7t-x-x-85">et</span><span 
class="ptmr7t-x-x-85">&#x00A0;al.</span></a><span 
class="ptmr7t-x-x-85">,</span><span 
class="ptmr7t-x-x-85">&#x00A0;</span><a 
href="#XNorden-Krichmar2007"><span 
class="ptmr7t-x-x-85">2007</span></a><span 
class="ptmr7t-x-x-85">),</span>
<span 
class="ptmr7t-x-x-85">(</span><a 
href="#XFu:08"><span 
class="ptmr7t-x-x-85">Fu</span>
<span 
class="ptmr7t-x-x-85">et</span><span 
class="ptmr7t-x-x-85">&#x00A0;al.</span></a><span 
class="ptmr7t-x-x-85">,</span><span 
class="ptmr7t-x-x-85">&#x00A0;</span><a 
href="#XFu:08"><span 
class="ptmr7t-x-x-85">2008</span></a><span 
class="ptmr7t-x-x-85">),</span>
<span 
class="ptmr7t-x-x-85">(</span><a 
href="#XHendrix2010"><span 
class="ptmr7t-x-x-85">Hendrix</span>
<span 
class="ptmr7t-x-x-85">et</span><span 
class="ptmr7t-x-x-85">&#x00A0;al.</span></a><span 
class="ptmr7t-x-x-85">,</span><span 
class="ptmr7t-x-x-85">&#x00A0;</span><a 
href="#XHendrix2010"><span 
class="ptmr7t-x-x-85">2010</span></a><span 
class="ptmr7t-x-x-85">),</span>
<span 
class="ptmr7t-x-x-85">(</span><a 
href="#XFriedlaender:12"><span 
class="ptmr7t-x-x-85">Friedl</span><span 
class="ptmr7t-x-x-85">änder</span>
<span 
class="ptmr7t-x-x-85">et</span><span 
class="ptmr7t-x-x-85">&#x00A0;al.</span></a><span 
class="ptmr7t-x-x-85">,</span><span 
class="ptmr7t-x-x-85">&#x00A0;</span><a 
href="#XFriedlaender:12"><span 
class="ptmr7t-x-x-85">2012</span></a><span 
class="ptmr7t-x-x-85">)</span> </p> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-20-"><td  style="white-space:nowrap; text-align:center;" id="TBL-2-20-1"  
class="td11"> <span 
class="ptmr7t-x-x-85">Cisa  </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-20-2"  
class="td11"><span 
class="ptmr7t-x-x-85">reftig</span><span 
class="ptmr7t-x-x-85">_613    </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-20-3"  
class="td11"><span 
class="zptmcm7t-x-x-85">31353   </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-20-4"  
class="td11"><span 
class="zptmcm7t-x-x-85">31949   </span></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-20-5"  
class="td11">
<!--l. 727--><p class="noindent" ><span 
class="ptmr7t-x-x-85">csa-mir-200,</span>
<span 
class="ptmr7t-x-x-85">csa-mir-141</span>              </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-20-6"  
class="td11">
<!--l. 728--><p class="noindent" ><span 
class="ptmr7t-x-x-85">NA</span>                                   </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-20-7"  
class="td11">
<!--l. 728--><p class="noindent" ><span 
class="ptmr7t-x-x-85">miRBase,</span>
<span 
class="ptmr7t-x-x-85">Homology</span>   </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-20-8"  
class="td11">
<!--l. 728--><p class="noindent" ><span 
class="ptmr7t-x-x-85">(</span><a 
href="#XFu:08"><span 
class="ptmr7t-x-x-85">Fu</span>
<span 
class="ptmr7t-x-x-85">et</span><span 
class="ptmr7t-x-x-85">&#x00A0;al.</span></a><span 
class="ptmr7t-x-x-85">,</span><span 
class="ptmr7t-x-x-85">&#x00A0;</span><a 
href="#XFu:08"><span 
class="ptmr7t-x-x-85">2008</span></a><span 
class="ptmr7t-x-x-85">)</span> </p> </td>
                                                               

                                                               
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-21-"><td  style="white-space:nowrap; text-align:center;" id="TBL-2-21-1"  
class="td11"> <span 
class="ptmr7t-x-x-85">Ciro  </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-21-2"  
class="td11"><span 
class="ptmr7t-x-x-85">7q                </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-21-3"  
class="td11"><span 
class="zptmcm7t-x-x-85">4153284</span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-21-4"  
class="td11"><span 
class="zptmcm7t-x-x-85">4156782</span></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-21-5"  
class="td11">
<!--l. 729--><p class="noindent" ><span 
class="ptmr7t-x-x-85">cin-mir-4006d,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4006c,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4001b-2,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4000i,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4006g,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4001e,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4001d,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4000g,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4006f,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4000h*,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4006b,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4001b-1,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4006e,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4001a-1,</span>
<span 
class="ptmb7t-x-x-85">cin-mir-4001a-2</span><span 
class="ptmr7t-x-x-85">,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4002,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4001h,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4000a-2,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4006a-2,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4006a-3,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4006a-1,</span>
<span 
class="ptmb7t-x-x-85">cin-mir-4006e</span><span 
class="ptmr7t-x-x-85">,</span>
<span 
class="ptmb7t-x-x-85">cin-mir-4001a-1</span><span 
class="ptmr7t-x-x-85">,</span>
<span 
class="ptmb7t-x-x-85">cin-mir-4006b</span><span 
class="ptmr7t-x-x-85">,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4000c,</span>
<span 
class="ptmb7t-x-x-85">cin-mir-4006e</span><span 
class="ptmr7t-x-x-85">,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4000b-2*,</span>
<span 
class="ptmb7t-x-x-85">cin-mir-4001a-1</span><span 
class="ptmr7t-x-x-85">,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4000b-1*,</span>
<span 
class="ptmb7t-x-x-85">cin-mir-4001a-2</span><span 
class="ptmr7t-x-x-85">,</span>
<span 
class="ptmb7t-x-x-85">cin-mir-4002</span><span 
class="ptmr7t-x-x-85">,</span>
<span 
class="ptmr7t-x-x-85">cin-mir-4000d*,</span>
<span 
class="ptmb7t-x-x-85">cin-mir-4001h</span><span 
class="ptmr7t-x-x-85">,</span>
<span 
class="ptmb7t-x-x-85">cin-mir-4006a-3</span><span 
class="ptmr7t-x-x-85">,</span>
<span 
class="ptmb7t-x-x-85">cin-mir-4006a-1</span><span 
class="ptmr7t-x-x-85">,</span>
<span 
class="ptmb7t-x-x-85">cin-mir-4000a-1</span>      </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-21-6"  
class="td11">
<!--l. 740--><p class="noindent" ><span 
class="ptmr7t-x-x-85">Elements marked with *</span>
<span 
class="ptmr7t-x-x-85">are                    identified</span>
<span 
class="ptmr7t-x-x-85">by  homology  strategies</span>
<span 
class="ptmr7t-x-x-85">at the same cluster, but in</span>
<span 
class="ptmr7t-x-x-85">another order reported by</span>
<span 
class="ptmr7t-x-x-85">miRBase.</span>                         </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-21-7"  
class="td11">
<!--l. 742--><p class="noindent" ><span 
class="ptmr7t-x-x-85">miRBase,</span>
<span 
class="ptmr7t-x-x-85">Homology</span>   </p></td><td  style="white-space:wrap; text-align:left;" id="TBL-2-21-8"  
class="td11">
<!--l. 742--><p class="noindent" ><span 
class="ptmr7t-x-x-85">(</span><a 
href="#XHendrix2010"><span 
class="ptmr7t-x-x-85">Hendrix</span>
<span 
class="ptmr7t-x-x-85">et</span><span 
class="ptmr7t-x-x-85">&#x00A0;al.</span></a><span 
class="ptmr7t-x-x-85">,</span><span 
class="ptmr7t-x-x-85">&#x00A0;</span><a 
href="#XHendrix2010"><span 
class="ptmr7t-x-x-85">2010</span></a><span 
class="ptmr7t-x-x-85">)</span> </p> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-22-"><td  style="white-space:nowrap; text-align:center;" id="TBL-2-22-1"  
class="td11">
</td></tr><tr 
class="cline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-2-23-"><td  style="white-space:nowrap; text-align:center;" id="TBL-2-23-1"  
class="td11">     </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-24-"><td  style="white-space:nowrap; text-align:center;" id="TBL-2-24-1"  
class="td11">     </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-25-"><td  style="white-space:nowrap; text-align:center;" id="TBL-2-25-1"  
class="td11">     </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-26-"><td  style="white-space:nowrap; text-align:center;" id="TBL-2-26-1"  
class="td11">     </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-27-"><td  style="white-space:nowrap; text-align:center;" id="TBL-2-27-1"  
class="td11">    </td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-27-2"  
class="td11">        </td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-27-3"  
class="td11">      </td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-27-4"  
class="td11">      </td><td  style="white-space:wrap; text-align:left;" id="TBL-2-27-5"  
class="td11">              </td><td  style="white-space:wrap; text-align:left;" id="TBL-2-27-6"  
class="td11">                </td><td  style="white-space:wrap; text-align:left;" id="TBL-2-27-7"  
class="td11">        </td><td  style="white-space:wrap; text-align:left;" id="TBL-2-27-8"  
class="td11"></td></tr>
</table></div>
</div>
  <hr class="figure" /><div class="figure" 
>
                                                               

                                                               
<a 
 id="x1-80198"></a>
                                                               

                                                               
<!--l. 750--><p class="noindent" >
<br /> </p><div class="caption" 
><span class="id">Fig.&#x00A0;8:  </span><span  
class="content">Absence/Presence  Matrix  of  miRNAs  families  along  Bilaterian  species.
<span 
class="ptmb7t-">Prot</span>: Protostomata, <span 
class="ptmb7t-">Brfl</span>: <span 
class="ptmri7t-">B. floridae</span>, <span 
class="ptmb7t-">Oidi</span>: <span 
class="ptmri7t-">O. dioica</span>, <span 
class="ptmb7t-">Dvex</span>: <span 
class="ptmri7t-">D. vexillum</span>, <span 
class="ptmb7t-">Ciin</span>: <span 
class="ptmri7t-">C.</span>
<span 
class="ptmri7t-">robusta</span>, <span 
class="ptmb7t-">Cisa</span>: <span 
class="ptmri7t-">C. savignyi</span>, <span 
class="ptmb7t-">Ciro</span>: <span 
class="ptmri7t-">C. robusta</span>, <span 
class="ptmb7t-">Sath</span>: <span 
class="ptmri7t-">S. thompsoni</span>, <span 
class="ptmb7t-">Mata</span>: <span 
class="ptmri7t-">M. oculata</span>,
<span 
class="ptmb7t-">Mlta</span>:  <span 
class="ptmri7t-">M.  occulta</span>,  <span 
class="ptmb7t-">Mlis</span>:  <span 
class="ptmri7t-">M.  occidentalis</span>,  <span 
class="ptmb7t-">Bosc</span>:  <span 
class="ptmri7t-">B.  schlosseri</span>,  <span 
class="ptmb7t-">Haro</span>:  <span 
class="ptmri7t-">H.  roretzi</span>,
<span 
class="ptmb7t-">Pema</span>:  <span 
class="ptmri7t-">P.  marinus</span>,  <span 
class="ptmb7t-">Dare</span>:  <span 
class="ptmri7t-">D.  rerio</span>,  <span 
class="ptmb7t-">Lach</span>:  <span 
class="ptmri7t-">L.  chalumnae</span>,  <span 
class="ptmb7t-">Xetr</span>:  <span 
class="ptmri7t-">X.  tropicalis  </span>and
<span 
class="ptmb7t-">Anca</span>: <span 
class="ptmri7t-">A. carolinensis</span>. </span></div><!--tex4ht:label?: x1-80198 -->
                                                               

                                                               
  </div><hr class="endfigure" />
  <hr class="figure" /><div class="figure" 
>
                                                               

                                                               
                                                               

                                                               
<a 
 id="x1-8020r9"></a>
<a 
 id="x1-8021"></a>
<span 
class="ptmb7t-x-x-85">Fig.</span><span 
class="ptmb7t-x-x-85">&#x00A0;9  </span><span 
class="ptmr7t-x-x-85">Dollo parsymony of</span>
<span 
class="ptmr7t-x-x-85">miRNAs families distribution</span>
<span 
class="ptmr7t-x-x-85">in some chordates genomes.</span>
<span 
class="ptmb7t-x-x-85">Prot</span><span 
class="ptmr7t-x-x-85">:  Protostomata,  </span><span 
class="ptmb7t-x-x-85">Brfl</span><span 
class="ptmr7t-x-x-85">:</span>
<span 
class="ptmri7t-x-x-85">B.  floridae</span><span 
class="ptmr7t-x-x-85">,  </span><span 
class="ptmb7t-x-x-85">Oidi</span><span 
class="ptmr7t-x-x-85">:  </span><span 
class="ptmri7t-x-x-85">O.</span>
<span 
class="ptmri7t-x-x-85">dioica</span><span 
class="ptmr7t-x-x-85">, </span><span 
class="ptmb7t-x-x-85">Dvex</span><span 
class="ptmr7t-x-x-85">: </span><span 
class="ptmri7t-x-x-85">D. vexillum</span><span 
class="ptmr7t-x-x-85">,</span>
<span 
class="ptmb7t-x-x-85">Ciro</span><span 
class="ptmr7t-x-x-85">:  </span><span 
class="ptmri7t-x-x-85">C.  robusta</span><span 
class="ptmr7t-x-x-85">,  </span><span 
class="ptmb7t-x-x-85">Cisa</span><span 
class="ptmr7t-x-x-85">:</span>
<span 
class="ptmri7t-x-x-85">C.  savignyi</span><span 
class="ptmr7t-x-x-85">,  </span><span 
class="ptmb7t-x-x-85">Sath</span><span 
class="ptmr7t-x-x-85">:  </span><span 
class="ptmri7t-x-x-85">S.</span>
<span 
class="ptmri7t-x-x-85">thompsoni</span><span 
class="ptmr7t-x-x-85">, </span><span 
class="ptmb7t-x-x-85">Mata</span><span 
class="ptmr7t-x-x-85">: </span><span 
class="ptmri7t-x-x-85">M. oculata</span><span 
class="ptmr7t-x-x-85">,</span>
<span 
class="ptmb7t-x-x-85">Mlta</span><span 
class="ptmr7t-x-x-85">:  </span><span 
class="ptmri7t-x-x-85">M.  occulta</span><span 
class="ptmr7t-x-x-85">,  </span><span 
class="ptmb7t-x-x-85">Mlis</span><span 
class="ptmr7t-x-x-85">:</span>
<span 
class="ptmri7t-x-x-85">M. occidentalis</span><span 
class="ptmr7t-x-x-85">, </span><span 
class="ptmb7t-x-x-85">Bosc</span><span 
class="ptmr7t-x-x-85">: </span><span 
class="ptmri7t-x-x-85">B.</span>
<span 
class="ptmri7t-x-x-85">schlosseri</span><span 
class="ptmr7t-x-x-85">, </span><span 
class="ptmb7t-x-x-85">Haro</span><span 
class="ptmr7t-x-x-85">: </span><span 
class="ptmri7t-x-x-85">H. roretzi</span><span 
class="ptmr7t-x-x-85">,</span>
<span 
class="ptmb7t-x-x-85">Pema</span><span 
class="ptmr7t-x-x-85">: </span><span 
class="ptmri7t-x-x-85">P. marinus</span><span 
class="ptmr7t-x-x-85">, </span><span 
class="ptmb7t-x-x-85">Dare</span><span 
class="ptmr7t-x-x-85">: </span><span 
class="ptmri7t-x-x-85">D.</span>
<span 
class="ptmri7t-x-x-85">rerio</span><span 
class="ptmr7t-x-x-85">, </span><span 
class="ptmb7t-x-x-85">Lach</span><span 
class="ptmr7t-x-x-85">: </span><span 
class="ptmri7t-x-x-85">L. chalumnae</span><span 
class="ptmr7t-x-x-85">,</span>
<span 
class="ptmb7t-x-x-85">Xetr</span><span 
class="ptmr7t-x-x-85">:  </span><span 
class="ptmri7t-x-x-85">X.  tropicalis  </span><span 
class="ptmr7t-x-x-85">and</span>
<span 
class="ptmb7t-x-x-85">Anca</span><span 
class="ptmr7t-x-x-85">: </span><span 
class="ptmri7t-x-x-85">A. carolinensis</span><span 
class="ptmr7t-x-x-85">. The</span>
<span 
class="ptmr7t-x-x-85">phylogenetic distribution of</span>
<span 
class="ptmr7t-x-x-85">this species was obtained from</span>
<span 
class="ptmr7t-x-x-85">(</span><a 
href="#XDelsuc:2017"><span 
class="ptmr7t-x-x-85">Delsuc et</span><span 
class="ptmr7t-x-x-85">&#x00A0;al.</span></a><span 
class="ptmr7t-x-x-85">,</span><span 
class="ptmr7t-x-x-85">&#x00A0;</span><a 
href="#XDelsuc:2017"><span 
class="ptmr7t-x-x-85">2017</span></a><span 
class="ptmr7t-x-x-85">;</span><span 
class="ptmr7t-x-x-85">&#x00A0;</span><a 
href="#XKocot:2018"><span 
class="ptmr7t-x-x-85">Kocot</span>
<span 
class="ptmr7t-x-x-85">et</span><span 
class="ptmr7t-x-x-85">&#x00A0;al.</span></a><span 
class="ptmr7t-x-x-85">,</span><span 
class="ptmr7t-x-x-85">&#x00A0;</span><a 
href="#XKocot:2018"><span 
class="ptmr7t-x-x-85">2018</span></a><span 
class="ptmr7t-x-x-85">).</span>
                                                               

                                                               
  </div><hr class="endfigure" />
  <h3 class="sectionHead"><span class="titlemark">3  </span> <a 
 id="x1-90003"></a>miRNAs and its role in development</h3>
<!--l. 786--><p class="noindent" >
</p>
  <h4 class="subsectionHead"><span class="titlemark">3.1  </span> <a 
 id="x1-100003.1"></a>miRNAs discovery and its role in development</h4>
<!--l. 788--><p class="noindent" >Both MicroRNAs (miRNAs or miRs) as well as MicroRNA offset RNA (moRNAs or
moRs) are developmentally regulated as shown during <span 
class="ptmri7t-">C. robusta </span>development (<a 
href="#XShi2009">Shi
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShi2009">2009</a>). In spite of the considerably higher abundance of miRs and miRs* in cells
than their corresponding abundance of moRs, all three small RNA types have been shown
to have regulatory roles in gene expression. Although a vast majority of miRNAs
remain to be studied, there are already many cases of well-studied miRNAs
(including many that are mentioned in this chapter that have been studied in
tunicates) that are known to target mRNAs, modulate their levels of expression, and
affect developmental processes both in plants and animals (<a 
href="#XZhao2018">Zhao et&#x00A0;al.</a>,&#x00A0;<a 
href="#XZhao2018">2018</a>).
Only recently two studies demonstrated for the first time that two moRs (viral
moR-rR1-3-5p and moR-21) could also modulate gene expression, and were
not merely the byproduct of miRNA biogenesis (<a 
href="#XUMBACH2010592">Umbach et&#x00A0;al.</a>,&#x00A0;<a 
href="#XUMBACH2010592">2010</a>;&#x00A0;<a 
href="#XZhao2016">Zhao
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XZhao2016">2016</a>).
</p><!--l. 802--><p class="noindent" >
</p>
  <h4 class="subsectionHead"><span class="titlemark">3.2  </span> <a 
 id="x1-110003.2"></a>Neuronal fate determination and regulation by miR-124</h4>
<!--l. 804--><p class="noindent" >The miRNA miR-124 is expressed in the nervous system of many animals, including
Drosophila (<a 
href="#XAboobaker18017">Aboobaker et&#x00A0;al.</a>,&#x00A0;<a 
href="#XAboobaker18017">2005</a>), <span 
class="ptmri7t-">C. elegans </span>(<a 
href="#XClark:2010">Clark et&#x00A0;al.</a>,&#x00A0;<a 
href="#XClark:2010">2010</a>) and humans
(<a 
href="#XSempere2004">Sempere et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSempere2004">2004</a>). As was first observed in <span 
class="ptmri7t-">in vitro </span>studies of mouse brain
cells, low expression of miR-124 was related to neural stem cell maintenance,
whereas high expression of miR-124 induced the differentiation of neuronal
cell types (<a 
href="#XCheng:2009qf">Cheng et&#x00A0;al.</a>,&#x00A0;<a 
href="#XCheng:2009qf">2009</a>). A regulative role of miR-124 in non-neural
vs. neural fate decisions was further investigated by embryonic experiments in
vivo (<a 
href="#XChen4943">Chen et&#x00A0;al.</a>,&#x00A0;<a 
href="#XChen4943">2011</a>) and by theoretical and in silico modeling analyses in
<span 
class="ptmri7t-">C. robusta </span>(<a 
href="#XChen2014">Chen et&#x00A0;al.</a>,&#x00A0;<a 
href="#XChen2014">2014</a>). These studies showed that miR-124 promotes
nervous system development by feedback interactions with Notch signaling. During
nervous system development of <span 
class="ptmri7t-">C. robusta</span>, cells in the dorsal and ventral midline
                                                               

                                                               
epidermis of the tailbud embryo either take an epidermal sensory neuron (ESN) or
peripheral nervous system (PNS) fate, a decision mediated by lateral inhibition
using a classical model of feedback loop regulation Notch-Delta signaling in
neighboring cells (<a 
href="#XCollier:1996">Collier et&#x00A0;al.</a>,&#x00A0;<a 
href="#XCollier:1996">1996</a>;&#x00A0;<a 
href="#XChen2014">Chen et&#x00A0;al.</a>,&#x00A0;<a 
href="#XChen2014">2014</a>). Cells that take an
ESN fate showed low expression of miR-124 presumably by Notch inhibition,
whereas cells that take a PNS fate expressed high levels of miR-124, which in the
latter case it was shown to target and repress non-neuronal genes (e.g. neuronal
repressors SCP1 and PTBP1) downstream of Notch signaling (<a 
href="#XChen4943">Chen et&#x00A0;al.</a>,&#x00A0;<a 
href="#XChen4943">2011</a>).
In addition, expression of miR-124 in larval epidermal cells was sufficient for
ectopic neural specification, which resembled mis-expression experiments using
Pou4, an important transcription factor for sensory neuron specification (<a 
href="#XChen4943">Chen
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XChen4943">2011</a>;&#x00A0;<a 
href="#XJoyceTang2013">Joyce Tang et&#x00A0;al.</a>,&#x00A0;<a 
href="#XJoyceTang2013">2013</a>). Whereas miR-124 targeting to SCP1 is thought
to have evolved in the vertebrates+tunicates, miR targeting to PTBP1 may be
conserved among bilaterians except for ecdysozoans (<a 
href="#XChen4943">Chen et&#x00A0;al.</a>,&#x00A0;<a 
href="#XChen4943">2011</a>) suggesting
that the miRNA regulatory logic in lateral inhibition models of Notch-Delta
signaling may have broader implications in other organisms yet to be studied (<a 
href="#XChen2014">Chen
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XChen2014">2014</a>). The research team also showed that miR-124 acted at the gastrula stage
and targeted other non-neural genes such as muscle determinant Macho-1 and
notochord determinant Brachyury to allow for ectodermal fate specification (<a 
href="#XChen4943">Chen
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XChen4943">2011</a>).
</p><!--l. 839--><p class="noindent" >
</p>
  <h5 class="subsubsectionHead"><span class="titlemark">3.2.1  </span> <a 
 id="x1-120003.2.1"></a>Muscle development and the polycistronic miR-1/miR-133 cluster</h5>
<!--l. 840--><p class="noindent" >A well-studied case of miRNA regulation in muscle development is the miR-1/miR-133
polycistronic cluster. Whereas miR-1 promotes differentiation of muscle, miR-133
promotes proliferation of muscle precursors (<a 
href="#XChen:2005yq">Chen et&#x00A0;al.</a>,&#x00A0;<a 
href="#XChen:2005yq">2005</a>). In the chordates, these
two miRNAs are encoded in an antisense direction in a relatively close localization (3-11
kb apart) within the gene <span 
class="ptmri7t-">mind bomb 1 </span>(MIB1), and transcribed as a single primary (i.e.
polycistronic) transcript. Except for Drosophila and ambulacrarians (i.e. echinoderms and
hemichordates), a close proximity of these two miRNAs has been documented in most
animal taxa suggesting some form of functional regulatory constraint of a condensed
miR-1/miR-133 cluster for the bilaterians (<a 
href="#XCampo-Paysaa:2011">Campo-Paysaa et&#x00A0;al.</a>,&#x00A0;<a 
href="#XCampo-Paysaa:2011">2011b</a>). During <span 
class="ptmri7t-">C.</span>
<span 
class="ptmri7t-">robusta </span>development, the polycistronic transcription can be detected in the nuclei of
presumptive tail muscle cells from the gastrula stage onward, and its transcription is
regulated by an <span 
class="zptmcm7t-">850 </span>bp sequence upstream of the transcript start site (<a 
href="#XKusakabe2013">Kusakabe
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XKusakabe2013">2013</a>). Differential expression of the two miRNAs in muscle tissues was only
detected in the adult, where body wall muscle expressed similar levels of miR-1 and
miR-133 and heart muscle expressed significantly higher levels of miR-1 (<a 
href="#XKusakabe2013">Kusakabe
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XKusakabe2013">2013</a>).
                                                               

                                                               
</p><!--l. 858--><p class="noindent" >
</p>
  <h5 class="subsubsectionHead"><span class="titlemark">3.2.2  </span> <a 
 id="x1-130003.2.2"></a>miRNA expression during oral siphon (OS) regeneration</h5>
<!--l. 859--><p class="noindent" >Three stages of regeneration have been proposed that reconstruct main events of
regeneration that match expected miRNAs expression profiles in the corresponding
timeframes (<a 
href="#XKnapp:2013">Knapp et&#x00A0;al.</a>,&#x00A0;<a 
href="#XKnapp:2013">2013</a>). The three phases correspond to: i. wound Healing, ii.
transition, and iii. re-development. The miRNA-mRNA transcriptional profiling using a
correlation network, correlated differential expression of mRNAs to miRNA
profiles during the three regeneration windows mentioned above in <span 
class="ptmri7t-">C. robusta </span>oral
siphon regeneration (<a 
href="#XSpina:2017">Spina et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSpina:2017">2017</a>). In the first phase, i.e. wound healing,
miRNA target clusters of miR 4178b-5p and miR 4_20211 were found to be
correlated to the differential expression of genes involved in the following Gene
Ontology (GO) term functional classifications: immune response, stress response and
apoptosis. In the second phase, i.e. transition, miR 4008c-5p, miR 4123-5p, miR
4178-5p, miR 2_15911, miR 4_20211, and miR 11_7539 were correlated and
known to target Wnt, TGFb and MAPK pathway genes that may be regulating the
proliferative state characteristic of this particular timeframe. In the third phase, i.e.
re-development, miR4008c-5p, miR 10_4533, and miR 11_6940 known to target ECM
peptidase inhibitors are correlated with the characteristic extracellular matrix
remodeling that occurs at the final phase of regeneration and which resembles the
original developmental processes. In contrast other miRs were found expressed
throughout the regenerative process. MiRNA miR 10_4533 known to target IGF and
IGFb was found expressed presumably regulating the proliferation of progenitor
cells. Also miR-9 was found expressed throughout regeneration and is known to
be essential for neural development and function, presumably by targeting and
regulating genes involved in cytoskeleton and cell cycle functions (<a 
href="#XGalderisi:2003rt">Galderisi
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGalderisi:2003rt">2003</a>;&#x00A0;<a 
href="#XMCBEATH2004483">McBeath et&#x00A0;al.</a>,&#x00A0;<a 
href="#XMCBEATH2004483">2004</a>), instead of targeting Notch or Hes-1 (<a 
href="#XSpina:2017">Spina
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSpina:2017">2017</a>).
</p><!--l. 888--><p class="noindent" >
</p>
  <h5 class="subsubsectionHead"><span class="titlemark">3.2.3  </span> <a 
 id="x1-140003.2.3"></a>miRNA expression during <span 
class="ptmri7t-">O. dioica </span>development</h5>
<!--l. 889--><p class="noindent" >A most thorough study of the miRNA repertoire expressed during development has
been published for the larvacean <span 
class="ptmri7t-">O. dioca </span>(<a 
href="#XFu:08">Fu et&#x00A0;al.</a>,&#x00A0;<a 
href="#XFu:08">2008</a>). Using a miRNA
array approach with <span 
class="zptmcm7t-">55 </span>candidate miRNAs and <span 
class="zptmcm7t-">10 </span>developmental stages for
analyses, some general patterns of miRNA occurrence emerged. MicroRNAs were
expressed throughout the life cycle of the animal, and were deposited in eggs as
maternal determinants for early zygotes. Expression of zygotic miRNAs, such
as miR-1487 and miR-1488, was observed starting in the blastula stage (1.5h
post fertilization). Most miRNAs analyzed showed developmental regulation
(for specific miRNAs that were differentially expressed at each stage see (<a 
href="#XFu:08">Fu
                                                               

                                                               
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XFu:08">2008</a>)), except for some such as miR-1497 that was expressed throughout all
stages (<a 
href="#XFu:08">Fu et&#x00A0;al.</a>,&#x00A0;<a 
href="#XFu:08">2008</a>). From this study, the first sex specific miRNAs were
revealed: miR-1478 was expressed in <span 
class="zptmcm7t-">6 </span>days old females in the oocytes, whereas
miR-1487/88 were expressed in <span 
class="zptmcm7t-">6 </span>days old males. Interestingly, the compact
genomes of <span 
class="ptmri7t-">O. dioica </span>showed one single copy of most miRNA loci, except for
miR-1490a, miR-1493, miR-1497d, and miR-1504 that were in two copies (<a 
href="#XFu:08">Fu
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XFu:08">2008</a>).
</p><!--l. 907--><p class="noindent" >
</p>
  <h3 class="sectionHead"><span class="titlemark">4  </span> <a 
 id="x1-150004"></a>Other ncRNAs associated with development</h3>
<!--l. 910--><p class="noindent" >
</p>
  <h4 class="subsectionHead"><span class="titlemark">4.1  </span> <a 
 id="x1-160004.1"></a>Yellow Crescent RNA</h4>
<!--l. 912--><p class="noindent" >Yellow crescent RNA, i.e. YC RNA, consists of an about 1.2 kb long polyadenylated
RNA, which can be present throughout the embryonic development of ascidians (<a 
href="#XSwalla1995">Swalla
and Jeffery</a>,&#x00A0;<a 
href="#XSwalla1995">1995</a>). Its name refers to the fact that in situ hybridization YC RNA is
localized in the yellow crescent region of one-cell zygotes. The YC transcripts are
actually already found in the cortex of unfertilized eggs, segregating with the
myoplasm to the yellow crescent after fertilization (<a 
href="#XSwalla1995">Swalla and Jeffery</a>,&#x00A0;<a 
href="#XSwalla1995">1995</a>).
Subsequently, most YC transcripts enter the primary muscle cell lineage after
cleavage and are also present in the secondary muscle cell lineage (<a 
href="#XSwalla1995">Swalla and
Jeffery</a>,&#x00A0;<a 
href="#XSwalla1995">1995</a>). YC RNA was first discovered in the club tunicate <span 
class="ptmri7t-">Styela clava </span>(<a 
href="#XSwalla1995">Swalla and
Jeffery</a>,&#x00A0;<a 
href="#XSwalla1995">1995</a>). As the presence of the 1.2-kb RNA in oocytes and early cleaving
embryos indicates that it is a maternal transcript, YC RNA is considered to be a
maternal RNA (<a 
href="#XSwalla1995">Swalla and Jeffery</a>,&#x00A0;<a 
href="#XSwalla1995">1995</a>). It is associated with the cytoskeleton and
segregates to the muscle cells during ascidian embryogenesis. Although the YC ORF
encodes for a putative polypeptide of <span 
class="zptmcm7t-">49 </span>amino acids, this protein is relatively
small and does not show any significant homology to any known proteins. As
the YC RNA shows various features indicating that it actually functions as an
RNA rather than as a protein coding molecule, it is considered to be a noncoding
RNA that may play an important role in growth and development (<a 
href="#XSwalla1995">Swalla and
Jeffery</a>,&#x00A0;<a 
href="#XSwalla1995">1995</a>).
                                                               

                                                               
</p><!--l. 934--><p class="noindent" >
</p>
  <h4 class="subsectionHead"><span class="titlemark">4.2  </span> <a 
 id="x1-170004.2"></a>MicroRNA-offset RNAs</h4>
<!--l. 936--><p class="noindent" >MicroRNA-offset RNAs, i.e. moRNAs, are about <span 
class="zptmcm7t-">20 </span>nucleotides long RNAs that lie
adjacent to pre-miRNAs. They can originate from both ends of these pre-miRNAs,
although prevalently they are derived from the 5&#8217; arm (<a 
href="#Xbortoluzzi2011">Bortoluzzi et&#x00A0;al.</a>,&#x00A0;<a 
href="#Xbortoluzzi2011">2011</a>). During a
study focused on identifying miRNAs in the simple chordate <span 
class="ptmri7t-">C. robusta </span>moRNAs were
first discovered (<a 
href="#XShi2009">Shi et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShi2009">2009</a>). Unexpectedly, half of the <span 
class="ptmri7t-">C. robusta </span>miRNA loci that
were detected in this study turned out to encode the previously uncharacterized small
RNAs, in addition to conventional miRNA and miRNA* products. This new class of
RNAs was hereafter referred to as &#8216;moRNAs&#8217;, for miRNA-offset RNAs. It became clear
that these moRNAs are probably produced by RNAse II-like processing and are observed,
like miRNAs, at specific developmental stages (<a 
href="#XShi2009">Shi et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShi2009">2009</a>). These results
and subsequent studies gave rise to the hypothesis that moRNAs represent a
new class of functional regulators whose qualitative alteration and/or expression
dysregulation might even impact human diseases (<a 
href="#Xbortoluzzi2011">Bortoluzzi et&#x00A0;al.</a>,&#x00A0;<a 
href="#Xbortoluzzi2011">2011</a>). Evidence
supporting this hypothesis is still fragmentary however. After the discovery in <span 
class="ptmri7t-">Ciona</span>,
moRNAs were also found in human cells by deep sequencing analysis. Hereby it
was reported that moRNAs from <span 
class="zptmcm7t-">78 </span>genomic loci were weakly expressed in the
prefrontal cortex (<a 
href="#XLangenberger2009">Langenberger et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLangenberger2009">2009</a>). Additional indications that moRNA
have a distinct function include the fact that some moRNAs are as conserved as
miRNAs and are in fact conserved across species to the extent that correlated with
expression level (<a 
href="#XShi2009">Shi et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShi2009">2009</a>). The expression level of certain moRNAs can even
be greater than for their corresponding miRNA (<a 
href="#XUmbach2010">Umbach and Cullen</a>,&#x00A0;<a 
href="#XUmbach2010">2010</a>).
Finally, it can be argued (<a 
href="#Xbortoluzzi2011">Bortoluzzi et&#x00A0;al.</a>,&#x00A0;<a 
href="#Xbortoluzzi2011">2011</a>) that it is likely that moRNAs
might represent a functional class of miRNA-related agents as moRNAs are
prevalently produced by the 5&#8217; arm of the precursor, independent of which arm
produces the most expressed mature miRNA (<a 
href="#XLangenberger2009">Langenberger et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLangenberger2009">2009</a>;&#x00A0;<a 
href="#XUmbach2010">Umbach
and Cullen</a>,&#x00A0;<a 
href="#XUmbach2010">2010</a>). What functions moRNAs may have, varies. For example,
moRNA expression was recorded in solid tumours, together with other small
RNAs (<a 
href="#XMeiri2010">Meiri et&#x00A0;al.</a>,&#x00A0;<a 
href="#XMeiri2010">2010</a>). In addition the fact that an 18-fold enrichment of
moRNAs was observed in the nucleus (<a 
href="#XTaft2010">Taft et&#x00A0;al.</a>,&#x00A0;<a 
href="#XTaft2010">2010</a>) indicates that at least
some moRNAs may have functions related to nuclear processes (<a 
href="#Xbortoluzzi2011">Bortoluzzi
et&#x00A0;al.</a>,&#x00A0;<a 
href="#Xbortoluzzi2011">2011</a>). Recently, an specific class of moRNAs (moRNA-21) has been
associated with post-transcriptional gene regulation, proliferation of vascular
smooth muscle cells (VSMC) and mediated gene down-regulation in a process
mediated by Ago2 (<a 
href="#XZhao2016">Zhao et&#x00A0;al.</a>,&#x00A0;<a 
href="#XZhao2016">2016</a>). Although these studies do provide good
indications, the potential functional roles that moRNAs can play, remain still largely
unknown.
                                                               

                                                               
</p><!--l. 975--><p class="noindent" >
</p>
  <h4 class="subsectionHead"><span class="titlemark">4.3  </span> <a 
 id="x1-180004.3"></a>Long Noncoding RNA RMST</h4>
<!--l. 977--><p class="noindent" >Long noncoding RNAs, i.e. lncRNAs, are abundantly found within mammalian
transcriptomes. One of the known groups of lncRNAs, includes the rhabdomyosarcoma
2-associated transcript (RMST), which is indispensable for neurogenesis (<a 
href="#XBogu2013">Ng
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBogu2013">2013</a>). Human RMST was shown to be responsible for the modulation of
neurogenesis as its expression is regulated by the transcriptional repressor REST while it
increases during neuronal differentiation (<a 
href="#XBogu2013">Ng et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBogu2013">2013</a>). Hereby it was found that
RMST is actually necessary for the binding of SOX2 to promoter regions of
neurogenic transcription factors. SOX2, a transcription factor known to regulate neural
fate, in combination with RMST were actually found to coregulate a large pool
of downstream genes implicated in neurogenesis, i.e. more than <span 
class="zptmcm7t-">1000 </span>genes
were differentially expressed upon RMST knockdown (<a 
href="#XBogu2013">Ng et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBogu2013">2013</a>). These
results illustrated the role of RMST as a transcriptional coregulator of SOX2
and a key player in the regulation of neural stem cell fate (<a 
href="#XBogu2013">Ng et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBogu2013">2013</a>). A
further confirmation of the importance of RMST came with the discovery of a
homologue of this lncRNA in the simple chordate <span 
class="ptmri7t-">D. vexillum</span>, i.e. the carpet sea-squirt
(<a 
href="#XVelandia-Huerto2016">Velandia-Huerto et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVelandia-Huerto2016">2016</a>). While homologues of &#8220;human&#8221; lncRNAs are rarely
found across all chordates due to their low levels of sequence conservation, a
plausible homolog of RMST <span 
class="zptmcm7t-">9</span>, the conserved region <span 
class="zptmcm7t-">9 </span>of the Rhabdomyosarcoma
2 associated transcript known for its interaction with SOX2, was found in <span 
class="ptmri7t-">D.</span>
<span 
class="ptmri7t-">vexillum</span>. Subsequently putative homologs were also found in the genomes of the
ascidians <span 
class="ptmri7t-">C. robusta</span>, <span 
class="ptmri7t-">C. savignyi </span>and <span 
class="ptmri7t-">B. schlosseri </span>and the Florida lancelet <span 
class="ptmri7t-">B.</span>
<span 
class="ptmri7t-">floridae</span>, illustrating that RMST lncRNA are thus conserved across chordates,
making them one of the best conserved lncRNAs known to date (<a 
href="#XVelandia-Huerto2016">Velandia-Huerto
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVelandia-Huerto2016">2016</a>).
</p><!--l. 1005--><p class="noindent" >
</p>
  <h4 class="subsectionHead"><span class="titlemark">4.4  </span> <a 
 id="x1-190004.4"></a>Splices-leader RNA</h4>
<!--l. 1007--><p class="noindent" >mRNA 5&#8217;leader trans-splicing is a mode of gene expression in which the 5&#8217; end of a
pre-mRNA is discarded and replaced by the 5&#8217; segment of a spliced leader (SL) RNA
(<a 
href="#XVandenberghe2001">Vandenberghe et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVandenberghe2001">2001</a>). Spliced-Leader RNAs, i.e. SL RNAs, hereby consist of a 5&#8217;
exon and a 3&#8217; intron with a conserved consensus 5&#8217; splice donor site at the exon-intron
boundary (<a 
href="#XGanot2004">Ganot et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGanot2004">2004</a>). SL RNA trans splicing has not only been described for
euglenoids, kinetoplastids, cnidarians, nematodes, and Platyhelminthes (<a 
href="#XGanot2004">Ganot
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGanot2004">2004</a>), but also for deuterostomes like the simple chordate <span 
class="ptmri7t-">C. robusta</span>
(<a 
href="#XVandenberghe2001">Vandenberghe et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVandenberghe2001">2001</a>) and the appendicularian <span 
class="ptmri7t-">O. dioica </span>(<a 
href="#XGanot2004">Ganot et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGanot2004">2004</a>).
Hereby <span 
class="ptmri7t-">O. dioica </span>was shown to not only trans-splice SL RNAs to mRNAs, as does <span 
class="ptmri7t-">C.</span>
                                                               

                                                               
<span 
class="ptmri7t-">robusta</span>, but also to use trans splicing in resolving polycistronic transcripts (<a 
href="#XGanot2004">Ganot
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGanot2004">2004</a>). During trans splicing, the capped SL RNA exon moiety is covalently linked
to the 5&#8217; ends of mRNAs, forming a leader sequence ranging from <span 
class="zptmcm7t-">16 </span>nt in <span 
class="ptmri7t-">C. robusta </span>to
<span 
class="zptmcm7t-">41 </span>nt in trypanosomatids (<a 
href="#XGanot2004">Ganot et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGanot2004">2004</a>). The role of SL trans-splicing is still
unknown in many cases. SL trans-splicing may potentially having functions varying from
the mediation of mRNA stability or translatability (<a 
href="#XMaroney1995">Maroney et&#x00A0;al.</a>,&#x00A0;<a 
href="#XMaroney1995">1995</a>) and the
resolution of polycistronic pre-mRNAs (<a 
href="#XAgabian1990">Agabian</a>,&#x00A0;<a 
href="#XAgabian1990">1990</a>;&#x00A0;<a 
href="#XBlumenthal1995">Blumenthal</a>,&#x00A0;<a 
href="#XBlumenthal1995">1995</a>), to the
production of functional mRNAs from RNA polymerase I transcripts (<a 
href="#XShuLee1997">Lee and der
Ploeg</a>,&#x00A0;<a 
href="#XShuLee1997">1997</a>).
</p><!--l. 1029--><p class="noindent" >
</p>
  <h3 class="sectionHead"><span class="titlemark">5  </span> <a 
 id="x1-200005"></a>Acknowledgments</h3>
<!--l. 1030--><p class="noindent" >This work and the computational analysis were partially supported by the equipment
donation from the German Academic Exchange Service-DAAD to the Faculty of
Science at the Universidad Nacional de Colombia. Also, by the computational
laboratory from Bioinformatics Group at Department of Computer Science and
Interdisciplinary Center for Bioinformatic at the Leipzig University. The comparative
analysis was partially supported by Colciencias (project no.&#x00A0;110165843196,
contract 571-2014). CAVH acknowledges the support by DAAD scholarship:
Forschungsstipendien-Promotionen in Deutschland, 2017/18 (Bewerbung 57299294).
CIBS acknowledges Universidad Nacional de Colombia for the time granted to write this
chapter.
</p><!--l. 1--><p class="noindent" >
</p>
  <h3 class="likesectionHead"><a 
 id="x1-210005"></a>References</h3>
<a 
 id="Q1-1-27"></a>
<!--l. 1--><p class="noindent" >
  </p><div class="thebibliography">
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XAboobaker18017"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Aboobaker,  A.&#x00A0;A.,  Tomancak,  P.,  Patel,  N.,  Rubin,  G.&#x00A0;M.,  and  Lai,  E.&#x00A0;C.
  (2005).  Drosophila micrornas exhibit diverse spatial expression patterns during
  embryonic  development.    <span 
class="ptmri7t-">Proceedings  of  the  National  Academy  of  Sciences</span>,
  102(50):18017&#8211;18022.
  </p>
                                                               

                                                               
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XAgabian1990"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Agabian, N. (1990).  Trans splicing of nuclear pre-mrnas.  <span 
class="ptmri7t-">Cell</span>, 61(7):1157 &#8211;
  1160.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XAmbrose:03a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Ambros, V., Bartel, B., Bartel, D.&#x00A0;P., Burge, C.&#x00A0;B., Carrington, J.&#x00A0;C., Chen, X.,
  Dreyfuss, G., Eddy, S.&#x00A0;R., Griffiths-Jones, S., Marshall, M., Matzke, M., Ruvkun,
  G., and Tuschl, T. (2003).  A uniform system for microRNA annotation.  <span 
class="ptmri7t-">RNA</span>,
  9:277&#8211;279.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XAmeres:13"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Ameres, S.&#x00A0;L. and Zamore, P.&#x00A0;D. (2013).  Diversifying microRNA sequence
  and function. <span 
class="ptmri7t-">Nature Rev. Mol. Cell Biol.</span>, 14:475&#8211;488.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XBartel:13"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Bartel, D.&#x00A0;P. (2009).  MicroRNA target recognition and regulatory functions.
  <span 
class="ptmri7t-">Cell</span>, 136:215&#8211;233.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XBashirullah1998"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Bashirullah,  A.,  Cooperstock,  R.&#x00A0;L.,  and  Lipshitz,  H.&#x00A0;D.  (1998).     RNA
  localization. <span 
class="ptmri7t-">Annu. Rev. Biochem</span>, 67:335&#8211;394.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XBentwich:05"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Bentwich,  I.,  Avniel,  A.&#x00A0;A.,  Karov,  Y.,  Aharonov,  R.,  Gilad,  S.,  Barad,  O.,
  Barzilai, A., Einat, P., Einav, U., Meiri, E., Sharon, E., Spector, Y., and Bentwich,
  Z.  (2005).   Identification  of  hundreds  of  conserved  and  nonconserved  human
  microRNAs. <span 
class="ptmri7t-">Nat.</span><span 
class="ptmri7t-">&#x00A0;Genet.</span>, 37:766&#8211;770.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XBerezikov:11"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Berezikov,  E.  (2011).    Evolution  of  microRNA  diversity  and  regulation  in
  animals. <span 
class="ptmri7t-">Nat Rev Genet.</span>, 12:846&#8211;860.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XBerezikov:06"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Berezikov,  E.,  Thuemmler,  F.,  van  Laake,  L.&#x00A0;W.,  Kondova,  I.,  Bontrop,  R.,
  Cuppen, E., and Plasterk, R.&#x00A0;H. (2006).  Diversity of microRNAs in human and
  chimpanzee brain. <span 
class="ptmri7t-">Nat Genet.</span>, 38:1375&#8211;1377.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XBlumenthal1995"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Blumenthal,  T.  (1995).     Trans-splicing  and  polycistronic  transcription  in
  caenohabditis elegans. <span 
class="ptmri7t-">Trends in Genetics</span>, 11(4):132 &#8211; 136.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="Xbortoluzzi2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Bortoluzzi, S., Biasiolo, M., and Bisognin, A. (2011). MicroRNA-offset RNAs
  (moRNAs): by-product spectators or functional players?   <span 
class="ptmri7t-">Trends in Molecular</span>
  <span 
class="ptmri7t-">Medicine</span>, 17(9):473&#8211;474.
                                                               

                                                               
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XBrunetti:2015"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Brunetti,  R.,  Gissi,  C.,  Pennati,  R.,  Caicci,  F.,  Gasparini,  F.,  and  Manni,
  L.  (2015).     Morphological  evidence  that  the  molecularly  determined  ciona
  intestinalis  type  a  and  type  b  are  different  species:  Ciona  robusta  and  ciona
  intestinalis.     <span 
class="ptmri7t-">Journal  of  Zoological  Systematics  and  Evolutionary  Research</span>,
  53(3):186&#8211;193.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XCampoPaysaa:11"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Campo-Paysaa, F., Sémon, M., Cameron, R.&#x00A0;A., Peterson, K.&#x00A0;J., and Schubert,
  M. (2011a).  MicroRNA complements in deuterostomes: origin and evolution of
  microRNAs. <span 
class="ptmri7t-">Evol Dev</span>, 13:15&#8211;27.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XCampo-Paysaa:2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Campo-Paysaa, F., Sémon, M., Cameron, R.&#x00A0;A., Peterson, K.&#x00A0;J., and Schubert,
  M. (2011b).   microrna complements in deuterostomes: origin and evolution of
  micrornas. <span 
class="ptmri7t-">Evolution &amp; Development</span>, 13(1):15&#8211;27.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XCerutti:06"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Cerutti, H. and Casas-Mollano, J.&#x00A0;A. (2006).  On the origin and functions of
  RNA-mediated silencing: from protists to man. <span 
class="ptmri7t-">Curr Genet</span>, 50:81&#8211;99.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XChen:2005yq"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Chen, J.-F., Mandel, E.&#x00A0;M., Thomson, J.&#x00A0;M., Wu, Q., Callis, T.&#x00A0;E., Hammond,
  S.&#x00A0;M.,  Conlon,  F.&#x00A0;L.,  and  Wang,  D.-Z.  (2005).     The  role  of  microRNA-1
  and microRNA-133 in skeletal muscle proliferation and differentiation.  <span 
class="ptmri7t-">Nature</span>
  <span 
class="ptmri7t-">Genetics</span>, 38:228&#8211;233.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XChen2014"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Chen,  J.&#x00A0;S.,  Gumbayan,  A.&#x00A0;M.,  Zeller,  R.&#x00A0;W.,  and  Mahaffy,  J.&#x00A0;M.  (2014).
  An   Expanded   Notch-Delta   Model   Exhibiting   Long-Range   Patterning   and
  Incorporating MicroRNA Regulation. <span 
class="ptmri7t-">PLoS Computational Biology</span>, 10(6).
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XChen4943"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Chen, J.&#x00A0;S., Pedro, M.&#x00A0;S., and Zeller, R.&#x00A0;W. (2011).  mir-124 function during
  ciona intestinalis neuronal development includes extensive interaction with the
  notch signaling pathway. <span 
class="ptmri7t-">Development</span>, 138(22):4943&#8211;4953.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XCheng:2009qf"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Cheng,  L.-C.,  Pastrana,  E.,  Tavazoie,  M.,  and  Doetsch,  F.  (2009).   mir-124
  regulates adult neurogenesis in the subventricular zone stem cell niche.  <span 
class="ptmri7t-">Nature</span>
  <span 
class="ptmri7t-">Neuroscience</span>, 12:399&#8211;408.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XClark:2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Clark,  A.&#x00A0;M.,  Goldstein,  L.&#x00A0;D.,  Tevlin,  M.,  Tavaré,  S.,  Shaham,  S.,  and
                                                               

                                                               
  Miska,  E.&#x00A0;A.  (2010).   The  microrna  mir-124  controls  gene  expression  in  the
  sensory  nervous  system  of  caenorhabditis  elegans.    <span 
class="ptmri7t-">Nucleic  Acids  Research</span>,
  38(11):3780&#8211;3793.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XCollier:1996"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Collier, J.&#x00A0;R., Monk, N.&#x00A0;A., Maini, P.&#x00A0;K., and Lewis, J.&#x00A0;H. (1996).   Pattern
  formation   by   lateral   inhibition   with   feedback:   a   mathematical   model   of
  delta-notch intercellular signalling. <span 
class="ptmri7t-">Journal of Theoretical Biology</span>, 183(4):429 &#8211;
  446.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XDai:09"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Dai,  Z.,  Chen,  Z.,  Ye,  H.,  Zhou,  L.,  Cao,  L.,  Wang,  Y.,  Peng,  S.,  and
  Chen,  L.  (2009).   Characterization  of  microRNAs  in  cephalochordates  reveals
  a   correlation   between   microRNA   repertoire   homology   and   morphological
  similarity in chordate evolution. <span 
class="ptmri7t-">Evol Dev.</span>, 11:41&#8211;49.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XDelsuc:2017"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Delsuc, F., Philippe, H., Tsagkogeorga, G., Simion, P., Tilak, M.-K., Turon,
  X.,  Lopez-Legentil,  S.,  Piette,  J.,  Lemaire,  P.,  and  Douzery,  E.  J.&#x00A0;P.  (2017).
  A  phylogenomic  framework  and  timescale  for  comparative  genomics  and
  evolutionary developmental biology of tunicates. <span 
class="ptmri7t-">bioRxiv</span>.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XDenoeud1381"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>   Denoeud,   F.,   Henriet,   S.,   Mungpakdee,   S.,   Aury,   J.-M.,   Da&#x00A0;Silva,   C.,
  Brinkmann, H., Mikhaleva, J., Olsen, L.&#x00A0;C., Jubin, C., Cañestro, C., Bouquet,
  J.-M., Danks, G., Poulain, J., Campsteijn, C., Adamski, M., Cross, I., Yadetie,
  F.,  Muffato,  M.,  Louis,  A.,  Butcher,  S.,  Tsagkogeorga,  G.,  Konrad,  A.,  Singh,
  S., Jensen, M.&#x00A0;F., Cong, E.&#x00A0;H., Eikeseth-Otteraa, H., Noel, B., Anthouard, V.,
  Porcel,  B.&#x00A0;M.,  Kachouri-Lafond,  R.,  Nishino,  A.,  Ugolini,  M.,  Chourrout,  P.,
  Nishida, H., Aasland, R., Huzurbazar, S., Westhof, E., Delsuc, F., Lehrach, H.,
  Reinhardt, R., Weissenbach, J., Roy, S.&#x00A0;W., Artiguenave, F., Postlethwait, J.&#x00A0;H.,
  Manak,  J.&#x00A0;R.,  Thompson,  E.&#x00A0;M.,  Jaillon,  O.,  Du&#x00A0;Pasquier,  L.,  Boudinot,  P.,
  Liberles, D.&#x00A0;A., Volff, J.-N., Philippe, H., Lenhard, B., Crollius, H.&#x00A0;R., Wincker,
  P., and Chourrout, D. (2010). Plasticity of animal genome architecture unmasked
  by rapid evolution of a pelagic tunicate. <span 
class="ptmri7t-">Science</span>, 330(6009):1381&#8211;1385.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XFriedlaender:12"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Friedländer, M.&#x00A0;R., Mackowiak, S.&#x00A0;D., Li, N., Chen, W., and Rajewsky, N.
  (2012). miRDeep2 accurately identifies known and hundreds of novel microRNA
  genes in seven animal clades. <span 
class="ptmri7t-">Nucleic Acids Res.</span>, 40:37&#8211;52.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XFu:08"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Fu, X., Adamski, M., and Thompson, E.&#x00A0;M. (2008). Altered miRNA repertoire
  in the simplified chordate, <span 
class="ptmri7t-">Oikopleura dioica</span>. <span 
class="ptmri7t-">Mol.</span><span 
class="ptmri7t-">&#x00A0;Biol.</span><span 
class="ptmri7t-">&#x00A0;Evol.</span>, 25:1067&#8211;1080.
  </p>
                                                               

                                                               
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XGalderisi:2003rt"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Galderisi, U., Jori, F.&#x00A0;P., and Giordano, A. (2003).  Cell cycle regulation and
  neural differentiation. <span 
class="ptmri7t-">Oncogene</span>, 22:5208&#8211;5219.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XGanot2004"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Ganot, P., Kallesøe, T., Reinhardt, R., Chourrout, D., and Thompson, E.&#x00A0;M.
  (2004).  Spliced-leader rna trans splicing in a chordate, oikopleura dioica, with a
  compact genome. <span 
class="ptmri7t-">Molecular and Cellular Biology</span>, 24(17):7795&#8211;7805.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XGriffiths-Jones2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Griffiths-Jones,  S.,  Hui,  J.  H.&#x00A0;L.,  Marco,  A.,  and  Ronshaugen,  M.  (2011).
  MicroRNA evolution by arm switching. <span 
class="ptmri7t-">EMBO reports</span>, 12(2):172&#8211;177.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XHeimberg:10"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Heimberg,  A.&#x00A0;M.,  Cowper-Sal<span 
class="zptmcm7y-">&#x22C5;</span>lari,  R.,  Sémon,  M.,  Donoghue,  P.&#x00A0;C.,  and
  Peterson,  K.&#x00A0;J.  (2010).    MicroRNAs  reveal  the  interrelationships  of  hagfish,
  lampreys, and gnathostomes and the nature of the ancestral vertebrate. <span 
class="ptmri7t-">Proc. Natl.</span>
  <span 
class="ptmri7t-">Acad. Sci. USA</span>, 107:19379&#8211;19383.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XHeimberg:08"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Heimberg,  A.&#x00A0;M.,  Sempere,  L.&#x00A0;F.,  Moy,  V.&#x00A0;N.,  Donoghue,  P.  C.&#x00A0;J.,  and
  Peterson,  K.  (2007).   MicroRNAs  and  the  advent  of  vertebrate  morphological
  complexity. <span 
class="ptmri7t-">Proc. Natl. Acad. Sci. USA</span>, 105:2946&#8211;2950.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XHendrix2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Hendrix,  D.,  Levine,  M.,  and  Shi,  W.  (2010).    miRTRAP,  a  computational
  method  for  the  systematic  identification  of  miRNAs  from  high  throughput
  sequencing data. <span 
class="ptmri7t-">Genome Biol</span>, 11(4):R39.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XHertel:12a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>   Hertel,   J.,   Bartschat,   S.,   Wintsche,   A.,   Otto,   C.,   The   Students   of   the
  Bioinformatics Computer Lab 2011, and Stadler, P.&#x00A0;F. (2012).  Evolution of the
  let-7 microRNA family. <span 
class="ptmri7t-">RNA Biology</span>, 9:231&#8211;241.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XHertel:06a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Hertel,  J.,  Lindemeyer,  M.,  Missal,  K.,  Fried,  C.,  Tanzer,  A.,  Flamm,  C.,
  Hofacker,  I.&#x00A0;L.,  Stadler,  P.&#x00A0;F.,  and  The  Students  of  Bioinformatics  Computer
  Labs  2004  and  2005  (2006).     The  expansion  of  the  metazoan  microRNA
  repertoire. <span 
class="ptmri7t-">BMC Genomics</span>, 7:15 [epub].
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XHertel:15a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Hertel,  J.  and  Stadler,  P.&#x00A0;F.  (2015).    The  expansion  of  animal  microRNA
  families revisited. <span 
class="ptmri7t-">Life</span>, 5:905&#8211;920.
  </p>
                                                               

                                                               
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XHolland2015"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Holland, L.&#x00A0;Z. (2015).   Genomics, evolution and development of amphioxus
  and tunicates: The Goldilocks principle.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XIkuta:2005"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Ikuta, T. and Saiga, H. (2005). Organization of hox genes in ascidians: Present,
  past, and future. <span 
class="ptmri7t-">Developmental Dynamics</span>, 233(2):382&#8211;389.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XJoyceTang2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Joyce  Tang,  W.,  Chen,  J.&#x00A0;S.,  and  Zeller,  R.&#x00A0;W.  (2013).     Transcriptional
  regulation of the peripheral nervous system in Ciona intestinalis.  <span 
class="ptmri7t-">Developmental</span>
  <span 
class="ptmri7t-">Biology</span>, 378(2):183&#8211;193.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XJue2016"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Jue, N.&#x00A0;K., Batta-Lona, P.&#x00A0;G., Trusiak, S., Obergfell, C., Bucklin, A., O&#8217;neill,
  M.&#x00A0;J., and O&#8217;neill, R.&#x00A0;J. (2016).  Rapid evolutionary rates and unique genomic
  signatures discovered in the first reference genome for the southern ocean salp,
  salpa  thompsoni  (Urochordata,  Thaliacea).    <span 
class="ptmri7t-">Genome  Biology  and  Evolution</span>,
  8(10):3171&#8211;3186.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XKeshavan2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>   Keshavan,   R.,   Virata,   M.,   Keshavan,   A.,   and   Zeller,   R.&#x00A0;W.   (2010).
  Computational  identification  of  Ciona  intestinalis  microRNAs.     <span 
class="ptmri7t-">Zoological</span>
  <span 
class="ptmri7t-">Science</span>, 27(2):162&#8211;170.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XKnapp:2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Knapp, D., Schulz, H., Rascon, C.&#x00A0;A., Volkmer, M., Scholz, J., Nacu, E., Le,
  M., Novozhilov, S., Tazaki, A., Protze, S., Jacob, T., Hubner, N., Habermann, B.,
  and Tanaka, E.&#x00A0;M. (2013).  Comparative transcriptional profiling of the axolotl
  limb  identifies  a  tripartite  regeneration-specific  gene  program.    <span 
class="ptmri7t-">PLOS  ONE</span>,
  8(5):1&#8211;20.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XKocot:2018"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Kocot,  K.&#x00A0;M.,  Tassia,  M.&#x00A0;G.,  Halanych,  K.&#x00A0;M.,  and  Swalla,  B.&#x00A0;J.  (2018).
  Phylogenomics  offers  resolution  of  major  tunicate  relationships.    <span 
class="ptmri7t-">Molecular</span>
  <span 
class="ptmri7t-">Phylogenetics and Evolution</span>, 121:166 &#8211; 173.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XKusakabe2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Kusakabe, R., Tani, S., Nishitsuji, K., Shindo, M., Okamura, K., Miyamoto, Y.,
  Nakai, K., Suzuki, Y., Kusakabe, T.&#x00A0;G., and Inoue, K. (2013).  Characterization
  of  the  compact  bicistronic  microRNA  precursor,  miR-1/miR-133,  expressed
  specifically in Ciona muscle tissues. <span 
class="ptmri7t-">Gene Expression Patterns</span>, 13(1-2):43&#8211;50.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XLangenberger2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Langenberger, D., Bermudez-Santana, C., Hertel, J., Hoffmann, S., Khaitovich,
                                                               

                                                               
  P., and Stadler, P.&#x00A0;F. (2009).  Evidence for human microrna-offset rnas in small
  rna sequencing data. <span 
class="ptmri7t-">Bioinformatics</span>, 25(18):2298&#8211;2301.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XLangenberger:10a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Langenberger,  D.,  Bermudez-Santana,  C.,  Stadler,  P.&#x00A0;F.,  and  Hoffmann,  S.
  (2010). Identification and classification of small RNAs in transcriptome sequence
  data. <span 
class="ptmri7t-">Pac.</span><span 
class="ptmri7t-">&#x00A0;Symp.</span><span 
class="ptmri7t-">&#x00A0;Biocomput.</span>, 15:80&#8211;87.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XLangenberger:13a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Langenberger,  D.,  Çakir,  M.&#x00A0;V.,  Hoffmann,  S.,  and  Stadler,  P.&#x00A0;F.  (2012).
  Dicer-processed  small  RNAs:  Rules  and  exceptions.   <span 
class="ptmri7t-">J.  Exp.  Zool:  Mol.  Dev.</span>
  <span 
class="ptmri7t-">Evol.</span>, 320:35&#8211;46.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XLee:07a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Lee, C.&#x00A0;T., Risom, T., and Strauss, W.&#x00A0;M. (2007).  Evolutionary conservation
  of microRNA regulatory circuits: an examination of microrna gene complexity
  and conserved microRNA-target interactions through metazoan phylogeny. <span 
class="ptmri7t-">DNA</span>
  <span 
class="ptmri7t-">Cell Biol.</span>, 26:209&#8211;218.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XShuLee1997"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Lee, M. G.-S. and der Ploeg, L. H. T.&#x00A0;V. (1997). Transcription of protein-coding
  genes in trypanosomes by rna polymerase i.   <span 
class="ptmri7t-">Annual Review of Microbiology</span>,
  51(1):463&#8211;489. PMID: 9343357.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XLegendre2005"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Legendre, M., Lambert, A., and Gautheret, D. (2005).  Profile-based detection
  of microRNA precursors in animal genomes. <span 
class="ptmri7t-">Bioinformatics</span>, 21(7):841&#8211;845.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XLiang:09"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Liang, H. and Li, W. (2009).  Lowly expressed human microrna genes evolve
  rapidly. <span 
class="ptmri7t-">Mol Biol Evol</span>, 26:1195&#8211;1198.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XLin:06"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Lin,  S.-L.,  Miller,  J.&#x00A0;D.,  and  Ying,  S.-Y.&#x00A0;Y.  (2006).    Intronic  MicroRNA
  (miRNA). <span 
class="ptmri7t-">J Biomed Biotechnol.</span>, 2006:26818.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XLu:08"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Lu, J., Shen, Y., Wu, Q., Kumar, S., He, B., Shi, S., Carthew, R.&#x00A0;W., Wang,
  S.&#x00A0;M.,  and  Wu,  C.&#x00A0;I.  (2008).    The  birth  and  death  of  microRNA  genes  in
  <span 
class="ptmri7t-">Drosophila</span>. <span 
class="ptmri7t-">Nat Genet.</span>, 40:351&#8211;355.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XLund:04"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Lund,  E.,  Güttinger,  S.,  Calado,  A.,  Dahlberg,  J.&#x00A0;E.,  and  Kutay,  U.  (2004).
  Nuclear export of microRNA precursors. <span 
class="ptmri7t-">Science</span>, 303:95&#8211;98.
  </p>
                                                               

                                                               
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XMarco:13"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Marco,  A.,  Ninova,  M.,  Ronshaugen,  M.,  and  Griffiths-Jones,  S.  (2013).
  Clusters of microRNAs emerge by new hairpins in existing transcripts.  <span 
class="ptmri7t-">Nucleic</span>
  <span 
class="ptmri7t-">Acids Res</span>, 41:7745&#8211;7752.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XMaroney1995"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Maroney,  P.&#x00A0;A.,  Denker,  J.&#x00A0;A.,  Darzynkiewicz,  E.,  Laneve,  R.,  and  Nilsen,
  T.&#x00A0;W. (1995). Most mrnas in the nematode ascaris lumbricoides are trans-spliced:
  a role for spliced leader addition in translational efficiency. <span 
class="ptmri7t-">RNA</span>, 1(7):714&#8211;723.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XMCBEATH2004483"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> McBeath, R., Pirone, D.&#x00A0;M., Nelson, C.&#x00A0;M., Bhadriraju, K., and Chen, C.&#x00A0;S.
  (2004).    Cell  shape,  cytoskeletal  tension,  and  rhoa  regulate  stem  cell  lineage
  commitment. <span 
class="ptmri7t-">Developmental Cell</span>, 6(4):483 &#8211; 495.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XMeiri2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Meiri, E., Levy, A., Benjamin, H., Ben-David, M., Cohen, L., Dov, A., Dromi,
  N., Elyakim, E., Yerushalmi, N., Zion, O., Lithwick-Yanai, G., and Sitbon, E.
  (2010).   Discovery of micrornas and other small rnas in solid tumors.   <span 
class="ptmri7t-">Nucleic</span>
  <span 
class="ptmri7t-">Acids Research</span>, 38(18):6234&#8211;6246.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XMeunier:12"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Meunier, J., Lemoine, F., Soumillon, M., Liechti, A., Weier, M., Guschanski,
  K.,  Hu,  H.,  Khaitovich,  P.,  and  Kaessmann,  H.  (2012).   Birth  and  expression
  evolution of mammalian microrna genes. <span 
class="ptmri7t-">Genome Res</span>, 23:34&#8211;45.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XMissal2005"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Missal, K., Rose, D., and Stadler, P.&#x00A0;F. (2005).   Non-coding RNAs in Ciona
  intestinalis. <span 
class="ptmri7t-">Bioinformatics</span>, 21(SUPPL. 2):77&#8211;78.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XNawrocki:13"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Nawrocki, E.&#x00A0;P. and Eddy, S.&#x00A0;R. (2013a).  Infernal 1.1: 100-fold faster RNA
  homology searches. <span 
class="ptmri7t-">Bioinformatics</span>, 29:2933&#8211;2935.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XNawrocki:2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Nawrocki,  E.&#x00A0;P.  and  Eddy,  S.&#x00A0;R.  (2013b).   Infernal  1.1:  100-fold  faster  rna
  homology searches. <span 
class="ptmri7t-">Bioinformatics</span>, 29(22):2933&#8211;2935.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XBogu2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Ng,  S.-Y.,  Bogu,  G.&#x00A0;K.,  Soh,  B.&#x00A0;S.,  and  Stanton,  L.&#x00A0;W.  (2013).   The  long
  noncoding rna rmst interacts with sox2 to regulate neurogenesis. <span 
class="ptmri7t-">Molecular Cell</span>,
  51(3):349&#8211;359.
  </p>
                                                               

                                                               
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XNiwa:07"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Niwa, R. and Slack, F.&#x00A0;J. (2007). The evolution of animal microRNA function.
  <span 
class="ptmri7t-">Curr Opin Genet Dev.</span>, 17:145&#8211;150.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XNorden-Krichmar2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Norden-Krichmar,  T.&#x00A0;M.,  Holtz,  J.,  Pasquinelli,  A.&#x00A0;E.,  and  Gaasterland,
  T.  (2007).     Computational  prediction  and  experimental  validation  of  Ciona
  intestinalis microRNA genes. <span 
class="ptmri7t-">BMC genomics</span>, 8(1):445.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XPasquinelli2003"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Pasquinelli, A.&#x00A0;E., McCoy, A., Jimenez, E., Salo, E., Ruvkun, G., Martindale,
  M.&#x00A0;Q.,   and   Baguna,   J.   (2003).       Expression   of   the   22   nucleotide   let-7
  heterochronic RNA throughout the Metazoa: a role in life history evolution? <span 
class="ptmri7t-">Evol.</span>
  <span 
class="ptmri7t-">Dev.</span>, 5(4):372&#8211;378.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XPasquinelli2000"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Pasquinelli,  a.&#x00A0;E.,  Reinhart,  B.&#x00A0;J.,  Slack,  F.,  Martindale,  M.&#x00A0;Q.,  Kuroda,
  M.&#x00A0;I., Maller, B., Hayward, D.&#x00A0;C., Ball, E.&#x00A0;E., Degnan, B., Müller, P., Spring,
  J.,  Srinivasan,  a.,  Fishman,  M.,  Finnerty,  J.,  Corbo,  J.,  Levine,  M.,  Leahy,  P.,
  Davidson, E., and Ruvkun, G. (2000). Conservation of the sequence and temporal
  expression of let-7 heterochronic regulatory RNA. <span 
class="ptmri7t-">Nature</span>, 408(6808):86&#8211;89.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XPeterson:09"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Peterson,  K.&#x00A0;J.,  Dietrich,  M.&#x00A0;R.,  and  McPeek,  M.&#x00A0;A.  (2009).   MicroRNAs
  and  metazoan  macroevolution:  insights  into  canalization,  complexity,  and  the
  Cambrian explosion. <span 
class="ptmri7t-">Bioessays</span>, 31:736&#8211;747.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XPrice:11"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Price, N., Cartwright, R.&#x00A0;A., Sabath, N., Graur, D., and Azevedo, R.&#x00A0;B. (2011).
  Neutral evolution of robustness in drosophila microRNA precursors.   <span 
class="ptmri7t-">Mol Biol</span>
  <span 
class="ptmri7t-">Evol</span>, 28:2115&#8211;2123.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XProchnik:07"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Prochnik, S.&#x00A0;E., Rokhsar, D.&#x00A0;S., and Aboobaker, A.&#x00A0;A. (2007). Evidence for a
  microRNA expansion in the bilaterian ancestor. <span 
class="ptmri7t-">Dev Genes Evol.</span>, 217:73&#8211;77.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XReinhart:2000mz"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Reinhart, B.&#x00A0;J., Slack, F.&#x00A0;J., Basson, M., Pasquinelli, A.&#x00A0;E., Bettinger, J.&#x00A0;C.,
  Rougvie,  A.&#x00A0;E.,  Horvitz,  H.&#x00A0;R.,  and  Ruvkun,  G.  (2000).    The  21-nucleotide
  let-7  rna  regulates  developmental  timing  in  caenorhabditis  elegans.    <span 
class="ptmri7t-">Nature</span>,
  403:901&#8211;906.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XSatou2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Satou, Y., Mineta, K., Ogasawara, M., Sasakura, Y., Shoguchi, E., Ueno, K.,
  Yamada, L., Matsumoto, J., Wasserscheid, J., Dewar, K., Wiley, G.&#x00A0;B., Macmil,
                                                               

                                                               
  S.&#x00A0;L.,  Roe,  B.&#x00A0;A.,  Zeller,  R.&#x00A0;W.,  Hastings,  K.&#x00A0;E.,  Lemaire,  P.,  Lindquist,  E.,
  Endo,  T.,  Hotta,  K.,  and  Inaba,  K.  (2008).    Improved  genome  assembly  and
  evidence-based  global  gene  model  set  for  the  chordate  ciona  intestinalis:  new
  insight into intron and operon populations. <span 
class="ptmri7t-">Genome Biology</span>, 9(10):R152.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XSempere:06"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Sempere,  L.&#x00A0;F.,  Cole,  C.&#x00A0;N.,  McPeek,  M.&#x00A0;A.,  and  Peterson,  K.&#x00A0;J.  (2006).
  The phylogenetic distribution of metazoan microRNAs: insights into evolutionary
  complexity and constraint. <span 
class="ptmri7t-">J Exp Zoolog B Mol Dev Evol.</span>, 306B:575&#8211;588.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XSempere2004"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Sempere, L.&#x00A0;F., Freemantle, S., Pitha-Rowe, I., Moss, E., Dmitrovsky, E., and
  Ambros, V. (2004).   Expression profiling of mammalian micrornas uncovers a
  subset  of  brain-expressed  micrornas  with  possible  roles  in  murine  and  human
  neuronal differentiation. <span 
class="ptmri7t-">Genome Biology</span>, 5(3):R13.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XShabalina:08"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Shabalina, S.&#x00A0;A. and Koonin, E.&#x00A0;V. (2008). Origins and evolution of eukaryotic
  RNA interference. <span 
class="ptmri7t-">Trends Ecol Evol.</span>, 23:578&#8211;587.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XShi2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Shi, W., Hendrix, D., Levine, M., and Haley, B. (2009). A distinct class of small
  RNAs arises from pre-miRNA&#8211;proximal regions in a simple chordate.   <span 
class="ptmri7t-">Nature</span>
  <span 
class="ptmri7t-">Structural &amp; Molecular Biology</span>, 16(2):183&#8211;189.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XSpina:2017"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Spina, E.&#x00A0;J., Guzman, E., Zhou, H., Kosik, K.&#x00A0;S., and Smith, W.&#x00A0;C. (2017).
  A microrna-mrna expression network during oral siphon regeneration in ciona.
  <span 
class="ptmri7t-">Development</span>, 144(10):1787&#8211;1797.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XSwalla1995"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Swalla,  B.&#x00A0;J.  and  Jeffery,  W.&#x00A0;R.  (1995).     A  maternal  RNA  localized  in
  the  yellow  crescent  is  segregated  to  the  larval  muscle  cells  during  ascidian
  development.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XTaft2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Taft,  R.&#x00A0;J.,  Pang,  K.&#x00A0;C.,  Mercer,  T.&#x00A0;R.,  Dinger,  M.,  and  Mattick,  J.&#x00A0;S.
  (2010).    Non-coding  rnas:  regulators  of  disease.    <span 
class="ptmri7t-">The  Journal  of  Pathology</span>,
  220(2):126&#8211;139.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XTanzer:10a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Tanzer,  A.,  Riester,  M.,  Hertel,  J.,  Bermudez-Santana,  C.&#x00A0;I.,  Gorodkin,  J.,
  Hofacker, I.&#x00A0;L., and Stadler, P.&#x00A0;F. (2010). Evolutionary genomics of microRNAs
  and their relatives.  In Caetano-Anolles, G., editor, <span 
class="ptmri7t-">Evolutionary Genomics and</span>
  <span 
class="ptmri7t-">Systems Biology</span>, pages 295&#8211;327. Wiley-Blackwell, Hoboken, NJ.
                                                               

                                                               
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XTanzer:04a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Tanzer,  A.  and  Stadler,  P.&#x00A0;F.  (2004).    Molecular  evolution  of  a  microRNA
  cluster. <span 
class="ptmri7t-">J.</span><span 
class="ptmri7t-">&#x00A0;Mol.</span><span 
class="ptmri7t-">&#x00A0;Biol.</span>, 339:327&#8211;335.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XTerai2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Terai,  G.,  Okida,  H.,  Asai,  K.,  and  Mituyama,  T.  (2012).     Prediction  of
  Conserved  Precursors  of  miRNAs  and  Their  Mature  Forms  by  Integrating
  Position-Specific Structural Features. <span 
class="ptmri7t-">PLoS ONE</span>, 7(9):1&#8211;11.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XUmbach2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Umbach,  J.&#x00A0;L.  and  Cullen,  B.&#x00A0;R.  (2010).     In-depth  analysis  of  kaposi&#8217;s
  sarcoma-associated  herpesvirus  microrna  expression  provides  insights  into  the
  mammalian microrna-processing machinery. <span 
class="ptmri7t-">Journal of Virology</span>, 84(2):695&#8211;703.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XUMBACH2010592"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Umbach,  J.&#x00A0;L.,  Strelow,  L.&#x00A0;I.,  Wong,  S.&#x00A0;W.,  and  Cullen,  B.&#x00A0;R.  (2010).
  Analysis  of  rhesus  rhadinovirus  micrornas  expressed  in  virus-induced  tumors
  from infected rhesus macaques. <span 
class="ptmri7t-">Virology</span>, 405(2):592 &#8211; 599.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XVandenberghe2001"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Vandenberghe,  A.&#x00A0;E.,  Meedel,  T.&#x00A0;H.,  and  Hastings,  K.&#x00A0;E.  (2001).    mrna
  5-leader trans-splicing in the chordates. <span 
class="ptmri7t-">Genes &amp; Development</span>, 15(3):294&#8211;303.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XVelandia-Huerto2016"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Velandia-Huerto, C.&#x00A0;A., Gittenberger, A.&#x00A0;A., Brown, F.&#x00A0;D., Stadler, P.&#x00A0;F., and
  Bermudez-Santana,  C.&#x00A0;I.  (2016).   Automated  detection  of  ncrnas  in  the  draft
  genome sequence of a colonial tunicate: the carpet sea squirt didemnum vexillum.
  <span 
class="ptmri7t-">BMC Genomics</span>, 17:691.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XWang2017"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Wang,  K.,  Dantec,  C.,  Lemaire,  P.,  Onuma,  T.&#x00A0;A.,  and  Nishida,  H.  (2017).
  Genome-wide survey of miRNAs and their evolutionary history in the ascidian,
  Halocynthia roretzi. <span 
class="ptmri7t-">BMC Genomics</span>, 18(1):314.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XWheeler:09"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Wheeler,  B.&#x00A0;M.,  Heimberg,  A.&#x00A0;M.,  Moy,  V.&#x00A0;N.,  Sperling,  E.&#x00A0;A.,  Holstein,
  T.&#x00A0;W., Heber, S., and Peterson, K.&#x00A0;J. (2009).  The deep evolution of metazoan
  microRNAs. <span 
class="ptmri7t-">Evol. Dev.</span>, 11:50&#8211;68.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XWill2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Will, S., Reiche, K., Hofacker, I.&#x00A0;L., Stadler, P.&#x00A0;F., and Backofen, R. (2007).
  Inferring  noncoding  RNA  families  and  classes  by  means  of  genome-scale
  structure-based clustering. <span 
class="ptmri7t-">PLoS Computational Biology</span>, 3(4):680&#8211;691.
  </p>
                                                               

                                                               
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XYagi2003"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Yagi,  K.,  Satou,  Y.,  Mazet,  F.,  Shimeld,  S.&#x00A0;M.,  Degnan,  B.,  Rokhsar,  D.,
  Levine,  M.,  Kohara,  Y.,  and  Satoh,  N.  (2003).     A  genomewide  survey  of
  developmentally  relevant  genes  in  ciona  intestinalis.   <span 
class="ptmri7t-">Development  Genes  and</span>
  <span 
class="ptmri7t-">Evolution</span>, 213(5):235&#8211;244.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XZhao2016"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Zhao,  J.,  Schnitzler,  G.&#x00A0;R.,  Iyer,  L.&#x00A0;K.,  Aronovitz,  M.&#x00A0;J.,  Baur,  W.&#x00A0;E.,  and
  Karas,  R.&#x00A0;H.  (2016).     Microrna-offset  rna  alters  gene  expression  and  cell
  proliferation. <span 
class="ptmri7t-">PLOS ONE</span>, 11(6):1&#8211;16.
  </p>
  <p class="bibitem" ><span class="biblabel">
<a 
 id="XZhao2018"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>  Zhao,  Y.,  Cong,  L.,  and  Lukiw,  W.&#x00A0;J.  (2018).   Plant  and  animal  micrornas
  (mirnas)  and  their  potential  for  inter-kingdom  communication.    <span 
class="ptmri7t-">Cellular  and</span>
  <span 
class="ptmri7t-">Molecular Neurobiology</span>, 38(1):133&#8211;140.
</p>
  </div>
   
</body></html> 

                                                               


